package smart_descriptions;

sub get_smart_description {
	my $domain_name = shift;
	my $start_description_str = "";

my %smart_description_hash = (
'14_3_3' => 'SM00101____14-3-3 homologues____14-3-3 homologues mediates signal transduction by binding to phosphoserine-containing proteins. They are involved in growth factor signalling and also interact with MEK kinases.',
'35EXOc' => 'SM00474____3\'-5\' exonuclease____3\' -5\' exonuclease proofreading domain present in DNA polymerase I, Werner syndrome helicase, RNase D and other enzymes',
'4.1m' => 'SM00294____putative band 4.1 homologues\' binding motif____',
'53EXOc' => 'SM00475____5\'-3\' exonuclease____',
'A1pp' => 'SM00506____Appr-1"-p processing enzyme____Function determined by Martzen et al. Extended family detected by reciprocal PSI-BLAST searches (unpublished results, and Pehrson & Fuji).',
'A4_EXTRA' => 'SM00006____amyloid A4____amyloid A4 precursor of Alzheimers disease',
'AAA' => 'SM00382____ATPases associated with a variety of cellular activities____AAA - ATPases associated with a variety of cellular activities. This profile/alignment only detects a fraction of this vast family. The poorly conserved N-terminal helix is missing from the alignment.',
'AAA_PrkA' => 'SM00763____PrkA AAA domain____This is a family of PrkA bacterial and archaeal serine kinases approximately 630 residues long. This is the N-terminal AAA domain.',
'AAI' => 'SM00499____Plant lipid transfer protein / seed storage protein / trypsin-alpha amylase inhibitor domain family____',
'AARP2CN' => 'SM00785____AARP2CN (NUC121) domain____This domain is the central domain of AARP2. It is weakly similar to the GTP-binding domain of elongation factor TU PUBMED:15112237.',
'ACR' => 'SM00608____ADAM Cysteine-Rich Domain____',
'ACTIN' => 'SM00268____Actin____ACTIN subfamily of ACTIN/mreB/sugarkinase/Hsp70 superfamily',
'ADEAMc' => 'SM00552____tRNA-specific and double-stranded RNA adenosine deaminase (RNA-specific editase)____',
'ADF' => 'SM00102____Actin depolymerisation factor/cofilin -like domains____Severs actin filaments and binds to actin monomers.',
'AFOR_N' => 'SM00790____Aldehyde ferredoxin oxidoreductase, N-terminal domain____Enzymes of the aldehyde ferredoxin oxidoreductase (AOR) family PUBMED:9242907 contain a tungsten cofactor and an 4Fe4S cluster and catalyse the interconversion of aldehydes to carboxylates PUBMED:8672295. This family includes AOR, formaldehyde ferredoxin oxidoreductase (FOR), glyceraldehyde-3-phosphate ferredoxin oxidoreductase (GAPOR), all isolated from hyperthermophilic archea PUBMED:9242907; carboxylic acid reductase found in clostridia PUBMED:2550230; and hydroxycarboxylate viologen oxidoreductase from Proteus vulgaris, the sole member of the AOR family containing molybdenum PUBMED:8026480. GAPOR may be involved in glycolysis PUBMED:7721730, but the functions of the other proteins are not yet clear. AOR has been proposed to be the primary enzyme responsible for oxidising the aldehydes that are produced by the 2-keto acid oxidoreductases PUBMED:9275170.',
'AGTRAP' => 'SM00805____Angiotensin II, type I receptor-associated protein____This family consists of several angiotensin II, type I receptor-associated protein (AGTRAP) sequences. AGTRAP is known to interact specifically with the C-terminal cytoplasmic region of the angiotensin II type 1 (AT(1)) receptor to regulate different aspects of AT(1) receptor physiology. The function of this family is unclear.',
'AHS1' => 'SM00796____Allophanate hydrolase subunit 1____This domain represents subunit 1 of allophanate hydrolase (AHS1).',
'AHS2' => 'SM00797____Allophanate hydrolase subunit 2____This domain represents subunit 2 of allophanate hydrolase (AHS2).',
'AICARFT_IMPCHas' => 'SM00798____AICARFT/IMPCHase bienzyme____This is a family of bifunctional enzymes catalysing the last two steps in de novo purine biosynthesis. The bifunctional enzyme is found in both prokaryotes and eukaryotes. The second last step is catalysed by 5-aminoimidazole-4-carboxamide ribonucleotide formyltransferase (AICARFT), this enzyme catalyses the formylation of AICAR with 10-formyl-tetrahydrofolate to yield FAICAR and tetrahydrofolate. The last step is catalysed by IMP (Inosine monophosphate) cyclohydrolase (IMPCHase), cyclizing FAICAR (5-formylaminoimidazole-4-carboxamide ribonucleotide) to IMP. ',
'AIP3' => 'SM00806____Actin interacting protein 3____Aip3p/Bud6p is a regulator of cell and cytoskeletal polarity in Saccharomyces cerevisiae that was previously identified as an actin-interacting protein. Actin-interacting protein 3 (Aip3p) localizes at the cell cortex where cytoskeleton assembly must be achieved to execute polarized cell growth, and deletion of AIP3 causes gross defects in cell and cytoskeletal polarity. Aip3p localization is mediated by the secretory pathway, mutations in early- or late-acting components of the secretory apparatus lead to Aip3p mislocalization PUBMED:10679021.',
'AKAP_110' => 'SM00807____A-kinase anchor protein 110 kDa____This family consists of several mammalian protein kinase A anchoring protein 3 (PRKA3) or A-kinase anchor protein 110 kDa (AKAP 110) sequences. Agents that increase intracellular cAMP are potent stimulators of sperm motility. Anchoring inhibitor peptides, designed to disrupt the interaction of the cAMP-dependent protein kinase A (PKA) with A kinase-anchoring proteins (AKAPs), are potent inhibitors of sperm motility. PKA anchoring is a key biochemical mechanism controlling motility. AKAP110 shares compartments with both RI and RII isoforms of PKA and may function as a regulator of both motility- and head-associated functions such as capacitation and the acrosome reaction PUBMED:10319321. ',
'ALBUMIN' => 'SM00103____serum albumin____',
'AMA-1' => 'SM00815____Apical membrane antigen 1____Apical membrane antigen 1 (AMA-1) is a Plasmodium asexual blood-stage antigen. It has been suggested that positive selection operates on the AMA-1 gene in regions coding for antigenic sites.',
'AMOP' => 'SM00723____Adhesion-associated domain present in MUC4 and other proteins____',
'ANATO' => 'SM00104____Anaphylatoxin homologous domain____C3a, C4a and C5a anaphylatoxins are protein fragments generated enzymatically in serum during activation of complement molecules C3, C4, and C5. They induce smooth muscle contraction. These fragments are homologous to a three-fold repeat in fibulins.',
'ANK' => 'SM00248____ankyrin repeats____Ankyrin repeats are about 33 amino acids long and occur in at least four consecutive copies. They are involved in protein-protein interactions. The core of the repeat seems to be an helix-loop-helix structure.',
'ANX' => 'SM00335____Annexin repeats____',
'AP2' => 'SM00380____DNA-binding domain in plant proteins such as APETALA2 and EREBPs____',
'AP2Ec' => 'SM00518____AP endonuclease family 2____These endonucleases play a role in DNA repair. Cleave phosphodiester bonds at apurinic or apyrimidinic sites',
'APPLE' => 'SM00223____APPLE domain____Four-fold repeat in plasma kallikrein and coagulation factor XI. Factor XI apple 3 mediates binding to platelets. Factor XI apple 1 binds high-molecular-mass kininogen. Apple 4 in factor XI mediates dimer formation and binds to factor XIIa. Mutations in apple 4 cause factor XI deficiency, an inherited bleeding disorder.',
'ARF' => 'SM00177____ARF-like small GTPases; ARF, ADP-ribosylation factor____Ras homologues involved in vesicular transport. Activator of phospholipase D isoforms. Unlike Ras proteins they lack cysteine residues at their C-termini and therefore are unlikely to be prenylated. ARFs are N-terminally myristoylated. Contains ATP/GTP-binding motif (P-loop).',
'ARM' => 'SM00185____Armadillo/beta-catenin-like repeats____Approx. 40 amino acid repeat. Tandem repeats form superhelix of helices that is proposed to mediate interaction of beta-catenin with its ligands. Involved in transducing the Wingless/Wnt signal. In plakoglobin arm repeats bind alpha-catenin and N-cadherin.',
'AT_hook' => 'SM00384____DNA binding domain with preference for A/T rich regions____Small DNA-binding motif first described in the high mobility group non-histone chromosomal protein HMG-I(Y).',
'AWS' => 'SM00570____associated with SET domains____subdomain of PRESET',
'AXH' => 'SM00536____domain in Ataxins and HMG containing proteins____unknown function',
'A_amylase_inhib' => 'SM00783____Alpha amylase inhibitor____Alpha amylase inhibitor inhibits mammalian alpha-amylases specifically, by forming a tight stoichiometric 1:1 complex with alpha-amylase. The inhibitor has no action on plant and microbial alpha amylases.',
'Aamy' => 'SM00642____Alpha-amylase domain____',
'Aamy_C' => 'SM00632________',
'Ad_cyc_g-alpha' => 'SM00789____Adenylate cyclase G-alpha binding domain____This fungal domain is found in adenylate cyclase and interacts with the alpha subunit of heterotrimeric G proteins.',
'Adenylsucc_synt' => 'SM00788____Adenylosuccinate synthetase____Adenylosuccinate synthetase plays an important role in purine biosynthesis, by catalyzing the GTP-dependent conversion of IMP and aspartic acid to AMP. Adenylosuccinate synthetase has been characterized from various sources ranging from Escherichia coli (gene purA) to vertebrate tissues. In vertebrates, two isozymes are present - one involved in purine biosynthesis and the other in the purine nucleotide cycle. The crystal structure of adenylosuccinate synthetase from E. coli reveals that the dominant structural element of each monomer of the homodimer is a central beta-sheet of 10 strands. The first nine strands of the sheet are mutually parallel with right-handed crossover connections between the strands. The 10th strand is antiparallel with respect to the first nine strands. In addition, the enzyme has two antiparallel beta-sheets, comprised of two strands and three strands each, 11 alpha-helices and two short 3/10-helices. Further, it has been suggested that the similarities in the GTP-binding domains of the synthetase and the p21ras protein are an example of convergent evolution of two distinct families of GTP-binding proteins PUBMED:8244965. Structures of adenylosuccinate synthetase from Triticum aestivum and Arabidopsis thaliana when compared with the known structures from E. coli reveals that the overall fold is very similar to that of the E. coli protein PUBMED:10669609.',
'Agenet' => 'SM00743____Tudor-like domain present in plant sequences.____Domain in plant sequences with possible chromatin-associated functions.',
'Agglutinin' => 'SM00791________Amaranthus caudatus agglutinin or amaranthin is a lectin from the ancient South American crop, amaranth grain. Although its biological function is unknown, it has a high binding specificity for the methyl-glycoside of the T-antigen, found linked to serine or threonine residues of cell surface glycoproteins PUBMED:2271665. The protein is comprised of a homodimer, with each homodimer consisting of two beta-trefoil domains PUBMED:9334739.',
'Agouti' => 'SM00792____Agouti protein____The agouti protein regulates pigmentation in the mouse hair follicle producing a black hair with a subapical yellow band. A highly homologous protein agouti signal protein (ASIP) is present in humans and is expressed at highest levels in adipose tissue where it may play a role in energy homeostasis and possibly human pigmentation PUBMED:11837451, PUBMED:11833005.',
'AgrB' => 'SM00793____Accessory gene regulator B____The accessory gene regulator (agr) of Staphylococcus aureus is the central regulatory system that controls the gene expression for a large set of virulence factors. The arg locus consists of two transcripts: RNAII and RNAIII. RNAII encodes four genes (agrA, B, C, and D) whose gene products assemble a quorum sensing system. At low cell density, the agr genes are continuously expressed at basal levels. A signal molecule, autoinducing peptide (AIP), produced and secreted by the bacteria, accumulates outside of the cells. When the cell density increases and the AIP concentration reaches a threshold, it activates the agr response, i.e. activation of secreted protein gene expression and subsequent repression of cell wall-associated protein genes. AgrB and AgrD are essential for the production of the autoinducing peptide which functions as a signal for quorum sensing. AgrB is a transmembrane protein PUBMED:11195102. AgrB is involved in the proteolytic processing of AgrD and may have both proteolytic enzyme activity and a transporter facilitating the export of the processed AgrD peptide PUBMED:12122003. ',
'AgrD' => 'SM00794____Staphylococcal AgrD protein____This family consists of several AgrD proteins from many Staphylococcus species. The agr locus was initially described in Staphylococcus aureus as an element controlling the production of exoproteins implicated in virulence. Its pattern of action has been shown to be complex, upregulating certain extracellular toxins and enzymes expressed post-exponentially and repressing some exponential-phase surface components. AgrD encodes the precursor of the autoinducing peptide (AIP).The AIP derived from AgrD by the action of AgrB interacts with AgrC in the membrane to activate AgrA, which upregulates transcription both from promoter P2, amplifying the response, and from P3, initiating the production of a novel effector: RNAIII. In S. aureus, delta-hemolysin is the only translation product of RNA III and is not involved in the regulatory functions of the transcript, which is therefore the primary agent for modulating the expression of other operons controlled by agr PUBMED:11807079.',
'Agro_virD5' => 'SM00795____Agrobacterium VirD5 protein____The virD operon in Agrobacterium encodes a site-specific endonuclease, and a number of other poorly characterised products. This family represents the VirD5 protein.',
'Alpha-L-AF_C' => 'SM00813____Alpha-L-arabinofuranosidase C-terminus____This entry represents the C terminus (approximately 200 residues) of bacterial and eukaryotic alpha-L-arabinofuranosidase. This catalyses the hydrolysis of non-reducing terminal alpha-L-arabinofuranosidic linkages in L-arabinose-containing polysaccharides.',
'Alpha-amyl_C2' => 'SM00810____Alpha-amylase C-terminal beta-sheet domain____This entry represents the beta-sheet domain that is found in several alpha-amylases, usually at the C-terminus. This domain is organised as a five-stranded anti-parallel beta-sheet.',
'Alpha-mann_mid' => 'SM00872____Alpha mannosidase, middle domain____Members of this entry belong to the glycosyl hydrolase family 38, This domain, which is found in the central region adopts a structure consisting of three alpha helices, in an immunoglobulin/albumin-binding domain-like fold. The domain is predominantly found in the enzyme alpha-mannosidase PUBMED:12634058.',
'Alpha_L_fucos' => 'SM00812____Alpha-L-fucosidase____O-Glycosyl hydrolases (EC 3.2.1.-) are a widespread group of enzymes that hydrolyse the glycosidic bond between two or more carbohydrates, or between a carbohydrate and a non-carbohydrate moiety. A classification system for glycosyl hydrolases, based on sequence similarity, has led to the definition of 85 different families. This classification is available on the CAZy (CArbohydrate-Active EnZymes) web site PUBMED:. Because the fold of proteins is better conserved than their sequences, some of the families can be grouped in \'clans\'. Family 29 encompasses alpha-L-fucosidases, which is a lysosomal enzyme responsible for hydrolyzing the alpha-1,6-linked fucose joined to the reducing-end N-acetylglucosamine of the carbohydrate moieties of glycoproteins. Deficiency of alpha-L-fucosidase results in the lysosomal storage disease fucosidosis.',
'Alpha_TIF' => 'SM00814____Alpha trans-inducing protein (Alpha-TIF)____Alpha-TIF (VP16) from Herpes Simplex virus is an essential tegument protein involved in the transcriptional activation of viral immediate early (IE) promoters (alpha genes) during the lytic phase of viral infection. VP16 associates with cellular transcription factors to enhance transcription rates, including the general transcription factor TFIIB and the transcriptional coactivator PC4. The N-terminal residues of VP16 confer specificity for the IE genes, while the C-terminal residues are responsible for transcriptional activation. Within the C-terminal region are two activation regions that can independently and cooperatively activate transcription. VP16 forms a transcriptional regulatory complex with two cellular proteins, the POU-domain transcription factor Oct-1 and the cell-proliferation factor HCF-1. VP16 is an alpha/beta protein with an unusual fold. Other transcription factors may have a similar topology.',
'Alpha_adaptinC2' => 'SM00809____Adaptin C-terminal domain____Adaptins are components of the adaptor complexes which link clathrin to receptors in coated vesicles. Clathrin-associated protein complexes are believed to interact with the cytoplasmic tails of membrane proteins, leading to their selection and concentration. Gamma-adaptin is a subunit of the golgi adaptor. Alpha adaptin is a heterotetramer that regulates clathrin-bud formation. The carboxyl-terminal appendage of the alpha subunit regulates translocation of endocytic accessory proteins to the bud site. This Ig-fold domain is found in alpha, beta and gamma adaptins and consists of a beta-sandwich containing 7 strands in 2 beta-sheets in a greek-key topology PUBMED:10430869, PUBMED:12176391. The adaptor appendage contains an additional N-terminal strand.',
'Alpha_kinase' => 'SM00811____Alpha-kinase family____This family is a novel family of eukaryotic protein kinase catalytic domains, which have no detectable similarity to conventional kinases. The family contains myosin heavy chain kinases and Elongation Factor-2 kinase and a bifunctional ion channel. This family is known as the alpha-kinase family. The structure of the kinase domain revealed unexpected similarity to eukaryotic protein kinases in the catalytic core as well as to metabolic enzymes with ATP-grasp domains.',
'Amb_V_allergen' => 'SM00816____Amb V Allergen____Amb V is an Ambrosia sp (ragweed) pollen allergen. Amb t V has been shown to contain a C-terminal helix as the major T cell epitope. Free sulphhydryl groups also play a major role in the T cell recognition of cross-reactivity T cell epitopes within these related allergens. ',
'Amb_all' => 'SM00656________',
'Amelin' => 'SM00817____Ameloblastin precursor (Amelin)____This family consists of several mammalian Ameloblastin precursor (Amelin) proteins. Matrix proteins of tooth enamel consist mainly of amelogenin but also of non-amelogenin proteins, which, although their volumetric percentage is low, have an important role in enamel mineralisation. One of the non-amelogenin proteins is ameloblastin, also known as amelin and sheathlin. Ameloblastin (AMBN) is one of the enamel sheath proteins which is though to have a role in determining the prismatic structure of growing enamel crystals.',
'Amelogenin' => 'SM00818________Amelogenins, cell adhesion proteins, play a role in the biomineralisation of teeth. They seem to regulate formation of crystallites during the secretory stage of tooth enamel development and are thought to play a major role in the structural organisation and mineralisation of developing enamel. The extracellular matrix of the developing enamel comprises two major classes of protein: the hydrophobic amelogenins and the acidic enamelins. Circular dichroism studies of porcine amelogenin have shown that the protein consists of 3 discrete folding units: the N-terminal region appears to contain beta-strand structures, while the C-terminal region displays characteristics of a random coil conformation. Subsequent studies on the bovine protein have indicated the amelogenin structure to contain a repetitive beta-turn segment and a "beta-spiral" between Gln112 and Leu138, which sequester a (Pro, Leu, Gln) rich region. The beta-spiral offers a probable site for interactions with Ca2+ ions. Muatations in the human amelogenin gene (AMGX) cause X-linked hypoplastic amelogenesis imperfecta, a disease characterised by defective enamel. A 9bp deletion in exon 2 of AMGX results in the loss of codons for Ile5, Leu6, Phe7 and Ala8, and replacement by a new threonine codon, disrupting the 16-residue (Met1-Ala16) amelogenin signal peptide.',
'Ami_2' => 'SM00644________',
'Ami_3' => 'SM00646________',
'AraC_E_bind' => 'SM00871____Bacterial transcription activator, effector binding domain____This domain is found in the probable effector binding domain of a number of different bacterial transcription activators PUBMED:10802742 and is also present in some DNA gyrase inhibitors. The absence of a HTH motif in the DNA gyrase inhibitors is thought to indicate the fact that these do not bind DNA.',
'ArfGap' => 'SM00105____Putative GTP-ase activating proteins for the small GTPase, ARF____Putative zinc fingers with GTPase activating proteins (GAPs) towards the small GTPase, Arf. The GAP of ARD1 stimulates GTPase hydrolysis for ARD1 but not ARFs.',
'Asparaginase' => 'SM00870________Asparaginase, which is found in various plant, animal and bacterial cells, catalyses the deamination of asparagine to yield aspartic acid and an ammonium ion, resulting in a depletion of free circulatory asparagine in plasma PUBMED:3026924. The enzyme is effective in the treatment of human malignant lymphomas, which have a diminished capacity to produce asparagine synthetase: in order to survive, such cells absorb asparagine from blood plasma PUBMED:2407723, PUBMED:3379033 - if Asn levels have been depleted by injection of asparaginase, the lymphoma cells die.',
'Autotransporter' => 'SM00869____Autotransporter beta-domain____Secretion of protein products occurs by a number of different pathways in bacteria. One of these pathways known as the type IV pathway was first described for the IgA1 protease. The protein component that mediates secretion through the outer membrane is contained within the secreted protein itself, hence the proteins secreted in this way are called autotransporters. This family corresponds to the presumed integral membrane beta-barrel domain that transports the protein. This domain is found at the C-terminus of the proteins it occurs in. The N-terminus contains the variable passenger domain that is translocated across the membrane. Once the passenger domain is exported it is cleaved auto-catalytically in some proteins, in others a different peptidase is used and in some cases no cleavage occurs.',
'B3_4' => 'SM00873____B3/4 domain____This domain is found in tRNA synthetase beta subunits as well as in some non tRNA synthetase proteins.',
'B41' => 'SM00295____Band 4.1 homologues____Also known as ezrin/radixin/moesin (ERM) protein domains. Present in myosins, ezrin, radixin, moesin, protein tyrosine phosphatases. Plasma membrane-binding domain. These proteins play structural and regulatory roles in the assembly and stabilization of specialized plasmamembrane domains. Some PDZ domain containing proteins bind one or more of this family. Now includes JAKs.',
'B5' => 'SM00874____tRNA synthetase B5 domain____This domain is found in phenylalanine-tRNA synthetase beta subunits.',
'B561' => 'SM00665____Cytochrome b-561 / ferric reductase transmembrane domain.____Cytochrome b-561 recycles ascorbate for the generation of norepinephrine by dopamine-beta-hydroxylase in the chromaffin vesicles of the adrenal gland. It is a transmembrane heme protein with the two heme groups being bound to conserved histidine residues. A cytochrome b-561 homologue, termed Dcytb, is an iron-regulated ferric reductase in the duodenal mucosa. Other homologues of these are also likely to be ferric reductases. SDR2 is proposed to be important in regulating the metabolism of iron in the onset of neurodegenerative disorders.',
'BACK' => 'SM00875____BTB And C-terminal Kelch____The BACK domain is found juxtaposed to the BTB domain; they are separated by as little as two residues.',
'BAG' => 'SM00264____BAG domains, present in regulator of Hsp70 proteins____BAG domains, present in Bcl-2-associated athanogene 1 and silencer of death domains',
'BAH' => 'SM00439____Bromo adjacent homology domain____',
'BAR' => 'SM00721________',
'BASIC' => 'SM00520____Basic domain in HLH proteins of MYOD family____',
'BATS' => 'SM00876____Biotin and Thiamin Synthesis associated domain____Biotin synthase (BioB), , catalyses the last step of the biotin biosynthetic pathway. The reaction consists in the introduction of a sulphur atom into dethiobiotin. BioB functions as a homodimer PUBMED:12482614. Thiamin synthesis if a complex process involving at least six gene products (ThiFSGH, ThiI and ThiJ). Two of the proteins required for the biosynthesis of the thiazole moiety of thiamine (vitamin B(1)) are ThiG and ThiH (this entry) and form a heterodimerPUBMED:12650933. Both of these reactions are thought of involve the binding of co-factors, and both function as dimers PUBMED:12482614, PUBMED:12650933. This domain therefore may be involved in co-factor binding or dimerisation.',
'BBC' => 'SM00502____B-Box C-terminal domain____Coiled coil region C-terminal to (some) B-Box domains',
'BBOX' => 'SM00336____B-Box-type zinc finger____',
'BCL' => 'SM00337____BCL (B-Cell lymphoma); contains BH1, BH2 regions____(BH1, BH2, (BH3 (one helix only)) and not BH4(one helix only)). Involved in apoptosis regulation',
'BH4' => 'SM00265____BH4 Bcl-2 homology region 4____',
'BHL' => 'SM00411____bacterial (prokaryotic) histone like domain____',
'BID_1' => 'SM00634____Bacterial Ig-like domain (group 1)____',
'BID_2' => 'SM00635____Bacterial Ig-like domain 2____',
'BIR' => 'SM00238____Baculoviral inhibition of apoptosis protein repeat____Domain found in inhibitor of apoptosis proteins (IAPs) and other proteins. Acts as a direct inhibitor of caspase enzymes.',
'BMC' => 'SM00877________Bacterial microcompartments are primitive organelles composed entirely of protein subunits. The prototypical bacterial microcompartment is the carboxysome, a protein shell for sequestering carbon fixation reactions. These proteins for hexameric structure.',
'BON' => 'SM00749____bacterial OsmY and nodulation domain____',
'BPI1' => 'SM00328____BPI/LBP/CETP N-terminal domain____Bactericidal permeability-increasing protein (BPI) / Lipopolysaccharide-binding protein (LBP) / Cholesteryl ester transfer protein (CETP) N-terminal domain',
'BPI2' => 'SM00329____BPI/LBP/CETP C-terminal domain____Bactericidal permeability-increasing protein (BPI) / Lipopolysaccharide-binding protein (LBP) / Cholesteryl ester transfer protein (CETP) C-terminal domain',
'BRCT' => 'SM00292____breast cancer carboxy-terminal domain____',
'BRIGHT' => 'SM00501____BRIGHT, ARID (A/T-rich interaction domain) domain____DNA-binding domain containing a helix-turn-helix structure',
'BRK' => 'SM00592____domain in transcription and CHROMO domain helicases____',
'BRLZ' => 'SM00338____basic region leucin zipper____',
'BROMO' => 'SM00297____bromo domain____',
'BSD' => 'SM00751____domain in transcription factors and synapse-associated proteins____ ',
'BTB' => 'SM00225____Broad-Complex, Tramtrack and Bric a brac____Domain in Broad-Complex, Tramtrack and Bric a brac. Also known as POZ (poxvirus and zinc finger) domain. Known to be a protein-protein interaction motif found at the N-termini of several C2H2-type transcription factors as well as Shaw-type potassium channels. Known structure reveals a tightly intertwined dimer formed via interactions between N-terminal strand and helix structures. However in a subset of BTB/POZ domains, these two secondary structures appear to be missing. Be aware SMART predicts BTB/POZ domains without the beta1- and alpha1-secondary structures.',
'BTK' => 'SM00107____Bruton\'s tyrosine kinase Cys-rich motif____Zinc-binding motif containing conserved cysteines and a histidine. Always found C-terminal to PH domains (but not all PH domains are followed by BTK motifs). The crystal structure shows this motif packs against the PH domain. The PH+Btk module pair has been called the Tec homology (TH) region.',
'BTP' => 'SM00576____Bromodomain transcription factors and PHD domain containing proteins____subdomain of archael histone-like transcription factors',
'B_lectin' => 'SM00108____Bulb-type mannose-specific lectin____',
'Bac_DnaA_C' => 'SM00760____Bacterial dnaA protein helix-turn-helix domain____Could be involved in DNA-binding.',
'Biotin_carb_C' => 'SM00878____Biotin carboxylase C-terminal domain____Biotin carboxylase is a component of the acetyl-CoA carboxylase multi-component enzyme which catalyses the first committed step in fatty acid synthesis in animals, plants and bacteria. Most of the active site residues reported in reference are in this C-terminal domain.',
'BowB' => 'SM00269____Bowman-Birk type proteinase inhibitor____',
'Brix' => 'SM00879________The Brix domain is found in a number of eukaryotic proteins including SSF proteins from yeast and humans, Arabidopsis thaliana Peter Pan-like protein and several hypothetical proteins.',
'C1' => 'SM00109____Protein kinase C conserved region 1 (C1) domains (Cysteine-rich domains)____Some bind phorbol esters and diacylglycerol. Some bind RasGTP. Zinc-binding domains.',
'C1Q' => 'SM00110____Complement component C1q domain.____Globular domain found in many collagens and eponymously in complement C1q. When part of full length proteins these domains form a \'bouquet\' due to the multimerization of heterotrimers. The C1q fold is similar to that of tumour necrosis factor.',
'C2' => 'SM00239____Protein kinase C conserved region 2 (CalB)____Ca2+-binding motif present in phospholipases, protein kinases C, and synaptotagmins (among others). Some do not appear to contain Ca2+-binding sites. Particular C2s appear to bind phospholipids, inositol polyphosphates, and intracellular proteins. Unusual occurrence in perforin. Synaptotagmin and PLC C2s are permuted in sequence with respect to N- and C-terminal beta strands. SMART detects C2 domains using one or both of two profiles.',
'C345C' => 'SM00643____Netrin C-terminal Domain____',
'C4' => 'SM00111____C-terminal tandem repeated domain in type 4 procollagens____Duplicated domain in C-terminus of type 4 collagens. Mutations in alpha-5 collagen IV are associated with X-linked Alport syndrome.',
'C8' => 'SM00832________This domain contains 8 conserved cysteine residues, but this family only contains 7 of them to overlaps with other domains. It is found in disease-related proteins including von Willebrand factor, Alpha tectorin, Zonadhesin and Mucin.',
'CA' => 'SM00112____Cadherin repeats.____Cadherins are glycoproteins involved in Ca2+-mediated cell-cell adhesion. Cadherin domains occur as repeats in the extracellular regions which are thought to mediate cell-cell contact when bound to calcium.',
'CAD' => 'SM00266____Domains present in proteins implicated in post-mortem DNA fragmentation____',
'CADG' => 'SM00736____Dystroglycan-type cadherin-like domains.____Cadherin-homologous domains present in metazoan dystroglycans and alpha/epsilon sarcoglycans, yeast Axl2p and in a very large protein from magnetotactic bacteria. Likely to bind calcium ions.',
'CALCITONIN' => 'SM00113____calcitonin____This family is formed by calcitonin, the calcitonin gene-related peptide, and amylin. They are short polypeptide hormones. ',
'CAP10' => 'SM00672____Putative lipopolysaccharide-modifying enzyme.____',
'CARD' => 'SM00114____Caspase recruitment domain____Motif contained in proteins involved in apoptotic signalling. Mediates homodimerisation. Structure consists of six antiparallel helices arranged in a topology homologue to the DEATH and the DED domain.',
'CARP' => 'SM00673____Domain in CAPs (cyclase-associated proteins) and X-linked retinitis pigmentosa 2 gene product.____',
'CASH' => 'SM00722____ Domain present in carbohydrate binding proteins and sugar hydrolses____',
'CASc' => 'SM00115____Caspase, interleukin-1 beta converting enzyme (ICE) homologues____Cysteine aspartases that mediate programmed cell death (apoptosis). Caspases are synthesised as zymogens and activated by proteolysis of the peptide backbone adjacent to an aspartate. The resulting two subunits associate to form an (alpha)2(beta)2-tetramer which is the active enzyme. Activation of caspases can be mediated by other caspase homologues.',
'CBD_II' => 'SM00637________',
'CBD_IV' => 'SM00606____Cellulose Binding Domain Type IV____',
'CBF' => 'SM00521____CCAAT-Binding transcription Factor____',
'CBS' => 'SM00116____Domain in cystathionine beta-synthase and other proteins.____Domain present in all 3 forms of cellular life. Present in two copies in inosine monophosphate dehydrogenase, of which one is disordered in the crystal structure [3]. A number of disease states are associated with CBS-containing proteins including homocystinuria, Becker\'s and Thomsen disease.',
'CCP' => 'SM00032____Domain abundant in complement control proteins; SUSHI repeat; short complement-like repeat (SCR)____The complement control protein (CCP) modules (also known as short  consensus repeats SCRs or SUSHI repeats) contain approximately 60 amino  acid residues and have been identified in several proteins of the  complement system. A missense mutation in seventh CCP domain causes  deficiency of the b subunit of factor XIII. ',
'CENPB' => 'SM00674____Putative DNA-binding domain in centromere protein B, mouse jerky and transposases.____',
'CFEM' => 'SM00747________eight cysteine-containing domain present in fungal extracellular membrane proteins',
'CH' => 'SM00033____Calponin homology domain____Actin binding domains present in duplicate at the N-termini of spectrin-like proteins (including dystrophin, alpha-actinin). These domains cross-link actin filaments into bundles and networks. A calponin homology domain is predicted in yeasst Cdc24p.',
'CHAD' => 'SM00880________The CHAD domain is an alpha-helical domain functionally associated with some members of the adenylate cyclase family . It has conserved histidines that may chelate metals.',
'CHK' => 'SM00587____ZnF_C4 abd HLH domain containing kinases domain____subfamily of choline kinases',
'CHRD' => 'SM00754____A domain in the BMP inhibitor chordin and in microbial proteins.____',
'CHROMO' => 'SM00298____Chromatin organization modifier domain____',
'CLECT' => 'SM00034____C-type lectin (CTL) or carbohydrate-recognition domain (CRD)____Many of these domains function as calcium-dependent carbohydrate binding modules.',
'CLH' => 'SM00299____Clathrin heavy chain repeat homology____',
'CLIP' => 'SM00680____Clip or disulphide knot domain____Present in horseshoe crab proclotting enzyme N-terminal domain, Drosophila Easter and silkworm prophenoloxidase-activating enzyme.',
'CLa' => 'SM00035____CLUSTERIN alpha chain____',
'CLb' => 'SM00030____CLUSTERIN Beta chain____',
'CM_2' => 'SM00830____Chorismate mutase type II____Chorismate mutase, catalyses the conversion of chorismate to prephenate in the pathway of tyrosine and phenylalanine biosynthesis. This enzyme is negatively regulated by tyrosine, tryptophan and phenylalanine PUBMED:9642265, PUBMED:9497350. ',
'CNH' => 'SM00036____Domain found in NIK1-like kinases, mouse citron and yeast ROM1, ROM2____Unpublished observations.',
'CNX' => 'SM00037____Connexin homologues____Connexin channels participate in the regulation of signaling between  developing and differentiated cell types.',
'COLFI' => 'SM00038____Fibrillar collagens C-terminal domain____Found at C-termini of fibrillar collagens: Ephydatia muelleri procollagen EMF1alpha, vertebrate collagens alpha(1)III, alpha(1)II, alpha(2)V etc.',
'COLIPASE' => 'SM00023____Colipase____ Colipase is a protein that functions as a cofactor for pancreatic lipase, with which it forms a stoichiometric complex. It also binds to the bile-salt covered triacylglycerol interface thus allowing the enzyme to anchor itself to the water-lipid interface. Colipase is a small protein of approximately 100 amino-acid residues with five conserved disulfide bonds.',
'CPDc' => 'SM00577____catalytic domain of ctd-like phosphatases____',
'CRA' => 'SM00757____CT11-RanBPM____protein-protein interaction domain present in crown eukaryotes (plants, animals, fungi)',
'CRF' => 'SM00039____corticotropin-releasing factor____',
'CSF2' => 'SM00040____Granulocyte-macrophage colony-simulating factor (GM-CSF)____GM-CSF stimulates the development of and the cytotoxic  activity of white blood cells.',
'CSP' => 'SM00357____Cold shock protein domain____RNA-binding domain that functions as a RNA-chaperone in bacteria and is involved in regulating translation in eukaryotes. Contains sub-family of RNA-binding domains in the Rho transcription termination factor. ',
'CT' => 'SM00041____C-terminal cystine knot-like domain (CTCK)____The structures of transforming growth factor-beta (TGFbeta), nerve growth factor (NGF), platelet-derived growth factor (PDGF) and gonadotropin all form 2 highly twisted antiparallel pairs of beta-strands and contain three disulphide bonds. The domain is non-globular and little is conserved among these presumed homologues except for their cysteine residues. CT domains are predicted to form homodimers.',
'CTLH' => 'SM00668____C-terminal to LisH motif. ____Alpha-helical motif of unknown function. ',
'CTNS' => 'SM00679____Repeated motif present between transmembrane helices in cystinosin, yeast ERS1p, mannose-P-dolichol utilization defect 1, and other hypothetical proteins.____Function unknown, but likely to be associated with the glycosylation machinery.',
'CUB' => 'SM00042____Domain first found in C1r, C1s, uEGF, and bone morphogenetic protein.____This domain is found mostly among developmentally-regulated proteins. Spermadhesins contain only this domain.',
'CUE' => 'SM00546____Domain that may be involved in binding ubiquitin-conjugating enzymes (UBCs)____CUE domains also occur in two protein of the IL-1 signal transduction pathway, tollip and TAB2. Ponting (Biochem. J.) "Proteins of the Endoplasmic reticulum" (in press)',
'CULLIN' => 'SM00182____Cullin____',
'CW' => 'SM00605________',
'CY' => 'SM00043____Cystatin-like domain____Cystatins are a family of cysteine protease inhibitors that occur mainly as single domain proteins. However some extracellular proteins such as  kininogen, His-rich glycoprotein and fetuin also contain these domains.',
'CYCLIN' => 'SM00385____domain present in cyclins, TFIIB and Retinoblastoma____A helical domain present in cyclins and TFIIB (twice) and Retinoblastoma (once). A protein recognition domain functioning in cell-cycle and transcription control.',
'CYCc' => 'SM00044____Adenylyl- / guanylyl cyclase, catalytic domain____Present in two copies in mammalian adenylyl cyclases. Eubacterial homologues are known. Two residues (Asn, Arg) are thought to be involved in catalysis. These cyclases have important roles in a diverse range of cellular processes.',
'Calx_beta' => 'SM00237____Domains in Na-Ca exchangers and integrin-beta4____Domain in Na-Ca exchangers and integrin subunit beta4 (and some cyanobacterial proteins)',
'Cation_ATPase_N' => 'SM00831____Cation transporter/ATPase, N-terminus____This entry represents the conserved N-terminal region found in several classes of cation-transporting P-type ATPases, including those that transport H+, Na+, Ca2+, Na+/K+, and H+/K+. In the H+/K+- and Na+/K+-exchange P-ATPases, this domain is found in the catalytic alpha chain. In gastric H+/K+-ATPases, this domain undergoes reversible sequential phosphorylation inducing conformational changes that may be important for regulating the function of these ATPases PUBMED:12480547, PUBMED:12529322.',
'ChSh' => 'SM00300____Chromo Shadow Domain____',
'ChW' => 'SM00728____Clostridial hydrophobic, with a conserved W residue, domain.____',
'CheW' => 'SM00260____Two component signalling adaptor domain____',
'ChtBD1' => 'SM00270____Chitin binding domain____',
'ChtBD2' => 'SM00494____Chitin-binding domain type 2____',
'ChtBD3' => 'SM00495____Chitin-binding domain type 3____',
'Citrate_ly_lig' => 'SM00764____Citrate lyase ligase C-terminal domain____Proteins of this family contain the C-terminal domain of citrate lyase ligase EC:6.2.1.22.',
'CoA_binding' => 'SM00881____CoA binding domain____This domain has a Rossmann fold and is found in a number of proteins including succinyl CoA synthetases, malate and ATP-citrate ligases.',
'CoA_trans' => 'SM00882____Coenzyme A transferase____Coenzyme A (CoA) transferases belong to an evolutionary conserved family of enzymes catalyzing the reversible transfer of CoA from one carboxylic acid to another. They have been identified in many prokaryotes and in mammalian tissues. The bacterial enzymes are heterodimer of two subunits (A and B) of about 25 Kd each while eukaryotic SCOT consist of a single chain which is colinear with the two bacterial subunits.',
'CobW_C' => 'SM00833____ Cobalamin synthesis protein cobW C-terminal domain____CobW proteins are generally found proximal to the trimeric cobaltochelatase subunit CobN, which is essential for vitamin B12 (cobalamin) biosynthesis PUBMED:12869542. They contain a P-loop nucleotide-binding loop in the N-terminal domain and a histidine-rich region in the C-terminal portion suggesting a role in metal binding, possibly as an intermediary between the cobalt transport and chelation systems. CobW might be involved in cobalt reduction leading to cobalt(I) corrinoids. This entry represents the C-terminal domain found in CobW, as well as in P47K, a Pseudomonas chlororaphis protein needed for nitrile hydratase expression PUBMED:7765511. ',
'Cog4' => 'SM00762____COG4 transport protein____This region is found in yeast oligomeric golgi complex component 4 which is involved in ER to Golgi and intra Golgi transport.',
'Cpn10' => 'SM00883____Chaperonin 10 Kd subunit____The chaperonins are \'helper\' molecules required for correct folding and subsequent assembly of some proteins. These are required for normal cell growth, and are stress-induced, acting to stabilise or protect disassembled polypeptides under heat-shock conditions. Type I chaperonins present in eubacteria, mitochondria and chloroplasts require the concerted action of 2 proteins, chaperonin 60 (cpn60) and chaperonin 10 (cpn10). The 10 kDa chaperonin (cpn10 - or groES in bacteria) exists as a ring-shaped oligomer of between six to eight identical subunits, while the 60 kDa chaperonin (cpn60 - or groEL in bacteria) forms a structure comprising 2 stacked rings, each ring containing 7 identical subunits. These ring structures assemble by self-stimulation in the presence of Mg2+-ATP. The central cavity of the cylindrical cpn60 tetradecamer provides as isolated environment for protein folding whilst cpn-10 binds to cpn-60 and synchronizes the release of the folded protein in an Mg2+-ATP dependent manner. The binding of cpn10 to cpn60 inhibits the weak ATPase activity of cpn60.',
'Cu_FIST' => 'SM00412____Copper-Fist____binds DNA only in present of copper or silver',
'Cullin_Nedd8' => 'SM00884____Cullin protein neddylation domain____This is the neddylation site of cullin proteins which are a family of structurally related proteins containing an evolutionarily conserved cullin domain. With the exception of APC2, each member of the cullin family is modified by Nedd8 and several cullins function in Ubiquitin-dependent proteolysis, a process in which the 26S proteasome recognises and subsequently degrades a target protein tagged with K48-linked poly-ubiquitin chains. Cullins are molecular scaffolds responsible for assembling the ROC1/Rbx1 RING-based E3 ubiquitin ligases, of which several play a direct role in tumorigenesis. Nedd8/Rub1 is a small ubiquitin-like protein, which was originally found to be conjugated to Cdc53, a cullin component of the SCF (Skp1-Cdc53/CUL1-F-box protein) E3 Ub ligase complex in Saccharomyces cerevisiae, and Nedd8 modification has now emerged as a regulatory pathway of fundamental importance for cell cycle control and for embryogenesis in metazoans. The only identified Nedd8 substrates are cullins. Neddylation results in covalent conjugation of a Nedd8 moiety onto a conserved cullin lysine residue.',
'Cupin_1' => 'SM00835____Cupin____This family represents the conserved barrel domain of the \'cupin\' superfamily (\'cupa\' is the Latin term for a small barrel). This family contains 11S and 7S plant seed storage proteins, and germins. Plant seed storage proteins provide the major nitrogen source for the developing plant.',
'CxxC_CXXC_SSSS' => 'SM00834____ Putative regulatory protein____CxxC_CXXC_SSSS represents a region of about 41 amino acids found in a number of small proteins in a wide range of bacteria. The region usually begins with the initiator Met and contains two CxxC motifs separated by 17 amino acids. One protein in this entry has been noted as a putative regulatory protein, designated FmdB. Most proteins in this entry have a C-terminal region containing highly degenerate sequence. ',
'CysPc' => 'SM00230____Calpain-like thiol protease family.____Calpain-like thiol protease family (peptidase family C2). Calcium activated neutral protease (large subunit).',
'D5_N' => 'SM00885____D5 N terminal like____This domain is found in D5 proteins of DNA viruses and bacteriophage P4 DNA primases phages.',
'DAGKa' => 'SM00045____Diacylglycerol kinase accessory domain (presumed)____Diacylglycerol (DAG) is a second messenger that acts as a protein kinase C activator. DAG can be produced from the hydrolysis of phosphatidylinositol 4,5-bisphosphate (PIP2) by a phosphoinositide-specific phospholipase C and by the degradation of phosphatidylcholine (PC) by a phospholipase C or the concerted actions of phospholipase D and phosphatidate phosphohydrolase. This domain might either be an accessory domain or else contribute to the catalytic domain. Bacterial homologues are known.',
'DAGKc' => 'SM00046____Diacylglycerol kinase catalytic domain (presumed)____Diacylglycerol (DAG) is a second messenger that acts as a protein kinase C activator. DAG can be produced from the hydrolysis of phosphatidylinositol 4,5-bisphosphate (PIP2) by a phosphoinositide-specific phospholipase C and by the degradation of phosphatidylcholine (PC) by a phospholipase C or the concerted actions of phospholipase D and phosphatidate phosphohydrolase. This domain  is presumed to be the catalytic domain. Bacterial homologues areknown.',
'DALR_1' => 'SM00836____DALR anticodon binding domain____This all alpha helical domain is the anticodon binding domain of Arginyl tRNA synthetase. This domain is known as the DALR domain after characteristic conserved amino acids PUBMED:10447505.',
'DALR_2' => 'SM00840________This DALR domain is found in cysteinyl-tRNA-synthetases.',
'DAX' => 'SM00021____Domain present in Dishevelled and axin____Domain of unknown function.',
'DCD' => 'SM00767________DCD is a plant specific domain in proteins involved in development and programmed cell death. The domain is shared by several proteins in the Arabidopsis and the rice genomes, which otherwise show a different protein architecture. Biological studies indicate a role of these proteins in phytohormone response, embryo development and programmed cell death by pathogens or ozone.',
'DCX' => 'SM00537____Domain in the Doublecortin (DCX) gene product____Tandemly-repeated domain in doublin, the Doublecortin gene product. Proposed to bind tubulin. Doublecortin (DCX) is mutated in human X-linked neuronal migration defects.',
'DDT' => 'SM00571____domain in different transcription and chromosome remodeling factors____',
'DEATH' => 'SM00005____DEATH domain, found in proteins involved in cell death (apoptosis).____Alpha-helical domain present in a variety of proteins with apoptotic functions. Some (but not all) of these domains form homotypic and heterotypic dimers.',
'DED' => 'SM00031____Death effector domain____',
'DEFSN' => 'SM00048____Defensin/corticostatin family____Cysteine-rich domains that lyse bacteria, fungi and enveloped viruses by forming multimeric membrane-spanning channels.',
'DENN' => 'SM00799____Domain found in a variety of signalling proteins, always encircled by uDENN and dDENN____The DENN domain is found in a variety of signalling proteins involved in Rab-mediated processes or regulation of MAPKs signalling pathways. The DENN domain is always encircled on both sides by more divergent domains, called uDENN (for upstream DENN) and dDENN (for downstream DENN). The function of the DENN domain remains to date unclear, although it appears to represent a good candidate for a GTP/GDP exchange activity.',
'DEP' => 'SM00049____Domain found in Dishevelled, Egl-10, and Pleckstrin____Domain of unknown function present in signalling proteins that contain PH, rasGEF, rhoGEF, rhoGAP, RGS, PDZ domains. DEP domain in Drosophila dishevelled is essential to rescue planar polarity defects and induce JNK signalling (Cell 94, 109-118).',
'DEXDc' => 'SM00487____DEAD-like helicases superfamily____',
'DEXDc2' => 'SM00488________',
'DEXDc3' => 'SM00489________',
'DISIN' => 'SM00050____Homologues of snake disintegrins  ____Snake disintegrins inhibit the binding of ligands to integrin receptors. They contain a \'RGD\' sequence, identical to the recognition site of many adhesion proteins. Molecules containing both disintegrin and metalloprotease domains are known as ADAMs. ',
'DM' => 'SM00301____Doublesex DNA-binding motif____',
'DM10' => 'SM00676____Domains in hypothetical proteins in Drosophila, C. elegans and mammals. Occurs singly in some nucleoside diphosphate kinases.____',
'DM11' => 'SM00675____Domains in hypothetical proteins in Drosophila including 2 in CG15241 and CG9329.____',
'DM13' => 'SM00686____Domain present in fly proteins (CG14681, CG12492, CG6217), worm H06A10.1 and Arabidopsis thaliana MBG8.9.____',
'DM14' => 'SM00685____Repeats in fly CG4713, worm Y37H9A.3 and human FLJ20241.____',
'DM15' => 'SM00684____Tandem repeat in fly CG14066 (La related protein), human KIAA0731 and worm R144.7. Unknown function.____',
'DM16' => 'SM00683____Repeats in sea squirt COS41.4, worm R01H10.6, fly CG1126 etc.____',
'DM3' => 'SM00692____Zinc finger domain in CG10631, C. elegans LIN-15B and human P52rIPK.____',
'DM4_12' => 'SM00718____DM4/DM12 family of domains in Drosophila melanogaster proteins of unknown function.____',
'DM5' => 'SM00690____Domain of unknown function, currently peculiar to Drosophila.____',
'DM6' => 'SM00689____Cysteine-rich domain currently specific to Drosophila.____',
'DM7' => 'SM00688____Domain of unknown function in Drosophila CG15332, CG15333 and CG18293____',
'DM8' => 'SM00697____Repeats found in several Drosophila proteins.____',
'DM9' => 'SM00696____Repeats found in Drosophila proteins.____',
'DNaseIc' => 'SM00476____deoxyribonuclease I____Deoxyribonuclease I catalyzes the endonucleolytic cleavage of double-stranded DNA. The enzyme is secreted outside the cell and also involved in apoptosis in the nucleus. ',
'DPBB_1' => 'SM00837____Rare lipoprotein A (RlpA)-like double-psi beta-barrel____Rare lipoprotein A (RlpA) contains a conserved region that has the double-psi beta-barrel (DPBB) fold. The function of RlpA is not well understood, but it has been shown to act as a prc mutant suppressor in Escherichia coli. The DPBB fold is often an enzymatic domain. The members of this family are quite diverse, and if catalytic this family may contain several different functions. Another example of this domain is found in the N terminus of pollen allergen.',
'DSL' => 'SM00051____delta serrate ligand____',
'DSPc' => 'SM00195____Dual specificity phosphatase, catalytic domain____',
'DSRM' => 'SM00358____Double-stranded RNA binding motif____',
'DUSP' => 'SM00695____Domain in ubiquitin-specific proteases.____',
'DWA' => 'SM00523____Domain A in dwarfin family proteins____',
'DWB' => 'SM00524____Domain B in dwarfin family proteins____',
'DYNc' => 'SM00053____Dynamin, GTPase____Large GTPases that mediate vesicle trafficking. Dynamin participates in the endocytic uptake of receptors, associated ligands, and  plasma membrane following an exocytic event.',
'DZF' => 'SM00572____domain in DSRM or ZnF_C2H2 domain containing proteins____',
'Dabb' => 'SM00886____Stress responsive A/B Barrel Domain____The function of this domain is unknown, but it is upregulated in response to salt stress in Populus balsamifera (balsam poplar). It is also found at the C-terminus of a fructose 1,6-bisphosphate aldolase from Hydrogenophilus thermoluteolus.It is found in the pA01 plasmid, which encodes genes for molybdopterin uptake and degradation of plant alkaloid nicotine. The structure of one has been solved and the domain forms an alpha-beta barrel dimer. Although there is a clear duplication within the domain it is not obviously detectable in the sequence. ',
'DnaG_DnaB_bind' => 'SM00766____DNA primase DnaG DnaB-binding____DnaG_DnaB_bind defines a domain of primase required for functional interaction with DnaB that attracts primase to the replication fork. DnaG_DnaB_bind is responsible for the interaction between DnaG and DnaB. ',
'DnaJ' => 'SM00271____DnaJ molecular chaperone homology domain____',
'DoH' => 'SM00664____Possible catecholamine-binding domain present in a variety of eukaryotic proteins.____A predominantly beta-sheet domain present as a regulatory N-terminal domain in dopamine beta-hydroxylase, mono-oxygenase X and SDR2. Its function remains unknown at present (Ponting, Human Molecular Genetics, in press).',
'DysFC' => 'SM00694____Dysferlin domain, C-terminal region.____Domain of unknown function present in yeast peroxisomal proteins, dysferlin, myoferlin and hypothetical proteins. Due to an insertion of a dysferlin domain within a second dysferlin domain we have chosen to predict these domains in two parts: the N-terminal region and the C-terminal region.',
'DysFN' => 'SM00693____Dysferlin domain, N-terminal region.____Domain of unknown function present in yeast peroxisomal proteins, dysferlin, myoferlin and hypothetical proteins. Due to an insertion of a dysferlin domain within a second dysferlin domain we have chosen to predict these domains in two parts: the N-terminal region and the C-terminal region.',
'EAL' => 'SM00052____Putative diguanylate phosphodiesterase____Putative diguanylate phosphodiesterase, present in a variety of bacteria.',
'EB_dh' => 'SM00887____Ethylbenzene dehydrogenase____Eythylbenzene dehydrogenase is a heterotrimer of three subunits that catalyses the anaerobic degradation of hydrocarbons. The alpha subunit contains the catalytic centre as a Molybdenum cofactor-complex. This removes an electron-pair from the hydrocarbon and passes it along an electron transport system involving iron-sulphur complexes held in the beta subunit and a Haem b molecule contained in the gamma subunit. The electron-pair is then subsequently passed to an as yet unknown receiver. The enzyme is found in a variety of different bacteria.',
'EF1_GNE' => 'SM00888____EF-1 guanine nucleotide exchange domain____Translation elongation factors are responsible for two main processes during protein synthesis on the ribosome. EF1A (or EF-Tu) is responsible for the selection and binding of the cognate aminoacyl-tRNA to the A-site (acceptor site) of the ribosome. EF2 (or EF-G) is responsible for the translocation of the peptidyl-tRNA from the A-site to the P-site (peptidyl-tRNA site) of the ribosome, thereby freeing the A-site for the next aminoacyl-tRNA to bind. Elongation factors are responsible for achieving accuracy of translation and both EF1A and EF2 are remarkably conserved throughout evolution. Elongation factor EF1B (also known as EF-Ts or EF-1beta/gamma/delta) is a nucleotide exchange factor that is required to regenerate EF1A from its inactive form (EF1A-GDP) to its active form (EF1A-GTP). EF1A is then ready to interact with a new aminoacyl-tRNA to begin the cycle again. EF1B is more complex in eukaryotes than in bacteria, and can consist of three subunits: EF1B-alpha (or EF-1beta), EF1B-gamma (or EF-1gamma) and EF1B-beta (or EF-1delta). This entry represents the guanine nucleotide exchange domain of the beta (EF-1beta, also known as EF1B-alpha) and delta (EF-1delta, also known as EF1B-beta) chains of EF1B proteins from eukaryotes and archaea. The beta and delta chains have exchange activity, which mainly resides in their homologous guanine nucleotide exchange domains, found in the C-terminal region of the peptides. Their N-terminal regions may be involved in interactions with the gamma chain (EF-1gamma).',
'EFG_C' => 'SM00838____Elongation factor G C-terminus____This domain includes the carboxyl terminal regions of Elongation factor G, elongation factor 2 and some tetracycline resistance proteins and adopt a ferredoxin-like fold.',
'EFG_IV' => 'SM00889____Elongation factor G, domain IV____Translation elongation factors are responsible for two main processes during protein synthesis on the ribosome. EF1A (or EF-Tu) is responsible for the selection and binding of the cognate aminoacyl-tRNA to the A-site (acceptor site) of the ribosome. EF2 (or EF-G) is responsible for the translocation of the peptidyl-tRNA from the A-site to the P-site (peptidyl-tRNA site) of the ribosome, thereby freeing the A-site for the next aminoacyl-tRNA to bind. Elongation factors are responsible for achieving accuracy of translation and both EF1A and EF2 are remarkably conserved throughout evolution. Elongation factor EF2 (EF-G) is a G-protein. It brings about the translocation of peptidyl-tRNA and mRNA through a ratchet-like mechanism: the binding of GTP-EF2 to the ribosome causes a counter-clockwise rotation in the small ribosomal subunit; the hydrolysis of GTP to GDP by EF2 and the subsequent release of EF2 causes a clockwise rotation of the small subunit back to the starting position. This twisting action destabilises tRNA-ribosome interactions, freeing the tRNA to translocate along the ribosome upon GTP-hydrolysis by EF2. EF2 binding also affects the entry and exit channel openings for the mRNA, widening it when bound to enable the mRNA to translocate along the ribosome. EF2 has five domains. This entry represents domain IV found in EF2 (or EF-G) of both prokaryotes and eukaryotes. The EF2-GTP-ribosome complex undergoes extensive structural rearrangement for tRNA-mRNA movement to occur. Domain IV, which extends from the \'body\' of the EF2 molecule much like a lever arm, appears to be essential for the structural transition to take place.',
'EFh' => 'SM00054____EF-hand, calcium binding motif____EF-hands are calcium-binding motifs that occur at least in pairs. Links between disease states and genes encoding EF-hands, particularly the S100 subclass, are emerging. Each motif consists of a 12 residue loop flanked on either side by a 12 residue alpha-helix. EF-hands undergo a conformational change unpon binding calcium ions.',
'EGF' => 'SM00181____Epidermal growth factor-like domain.____',
'EGF_CA' => 'SM00179____Calcium-binding EGF-like domain____',
'EGF_Lam' => 'SM00180____Laminin-type epidermal growth factor-like domai____',
'EGF_like' => 'SM00001____EGF domain, unclasssified subfamily____',
'EH' => 'SM00027____Eps15 homology domain____Pair of EF hand motifs that recognise proteins containing Asn-Pro-Phe (NPF) sequences.',
'EKR' => 'SM00890____Domain of unknown function____EKR is a short, 33 residue, domain found in bacterial and some lower eukaryotic species which lies between a POR (pyruvate ferredoxin/flavodoxin oxidoreductase) and the 4Fe-4S binding domain Fer4. It contains a characteristic EKR sequence motif. The exact function of this domain is not known.',
'ELFV_dehydrog' => 'SM00839____Glutamate/Leucine/Phenylalanine/Valine dehydrogenase____Glutamate, leucine, phenylalanine and valine dehydrogenases are structurally and functionally related. They contain a Gly-rich region containing a conserved Lys residue, which has been implicated in the catalytic activity, in each case a reversible oxidative deamination reaction.',
'END' => 'SM00272____Endothelin____',
'ENDO3c' => 'SM00478____endonuclease III____includes endonuclease III (DNA-(apurinic or apyrimidinic site) lyase), alkylbase DNA glycosidases (Alka-family) and other DNA glycosidases',
'ENTH' => 'SM00273____Epsin N-terminal homology (ENTH) domain____',
'EPEND' => 'SM00026____Ependymins____ Ependymins are the predominant proteins in the cerebrospinal fluid (CSF) of teleost fish. They have been implicated in the neurochemistry of memory and neuronal regeneration. They are glycoproteins of about 200 amino acids that can bind calcium. Four cysteines are conserved that probably form disulfide bonds.',
'EPH_lbd' => 'SM00615____Ephrin receptor ligand binding domain____',
'ERCC4' => 'SM00891____ERCC4 domain____This entry represents a structural motif found in several DNA repair nucleases, such as Rad1/Mus81/XPF endonucleases, and in ATP-dependent helicases. The XPF/Rad1/Mus81-dependent nuclease family specifically cleaves branched structures generated during DNA repair, replication, and recombination, and is essential for maintaining genome stability. The nuclease domain architecture exhibits remarkable similarity to those of restriction endonucleases.',
'ETF' => 'SM00893____Electron transfer flavoprotein domain____Electron transfer flavoproteins (ETFs) serve as specific electron acceptors for primary dehydrogenases, transferring the electrons to terminal respiratory systems. They can be functionally classified into constitutive, "housekeeping" ETFs, mainly involved in the oxidation of fatty acids (Group I), and ETFs produced by some prokaryotes under specific growth conditions, receiving electrons only from the oxidation of specific substrates (Group II). ETFs are heterodimeric proteins composed of an alpha and beta subunit, and contain an FAD cofactor and AMP. ETF consists of three domains: domains I and II are formed by the N- and C-terminal portions of the alpha subunit, respectively, while domain III is formed by the beta subunit. Domains I and III share an almost identical alpha-beta-alpha sandwich fold, while domain II forms an alpha-beta-alpha sandwich similar to that of bacterial flavodoxins. FAD is bound in a cleft between domains II and III, while domain III binds the AMP molecule. Interactions between domains I and III stabilise the protein, forming a shallow bowl where domain II resides. This entry represents the N-terminal domain of both the alpha and beta subunits from Group I and Group II ETFs.',
'ETS' => 'SM00413____erythroblast transformation specific domain____variation of the helix-turn-helix motif',
'EXOIII' => 'SM00479________exonuclease domain in DNA-polymerase alpha and epsilon chain, ribonuclease T and other exonucleases',
'EZ_HEAT' => 'SM00567____E-Z type HEAT repeats____Present in subunits of cyanobacterial phycocyanin lyase, and other proteins. Probable scaffolding role.',
'Elong-fact-P_C' => 'SM00841____Elongation factor P, C-terminal____These nucleic acid binding domains are predominantly found in elongation factor P, where they adopt an OB-fold, with five beta-strands forming a beta-barrel in a Greek-key topology PUBMED:15210970.',
'Elp3' => 'SM00729____Elongator protein 3, MiaB family, Radical SAM____This superfamily contains MoaA, NifB, PqqE, coproporphyrinogen III oxidase, biotin synthase and MiaB families, and includes a representative in the eukaryotic elongator subunit, Elp-3. Some members of the family are methyltransferases. ',
'Endonuclease_NS' => 'SM00892____DNA/RNA non-specific endonuclease____A family of bacterial and eukaryotic endonucleases share the following characteristics: they act on both DNA and RNA, cleave double-stranded and single-stranded nucleic acids and require a divalent ion such as magnesium for their activity. An histidine has been shown to be essential for the activity of the Serratia marcescens nuclease. This residue is located in a conserved region which also contains an aspartic acid residue that could be implicated in the binding of the divalent ion.',
'Excalibur' => 'SM00894____Excalibur calcium-binding domain____Extracellular Ca2+-dependent nuclease YokF from Bacillus subtilis and several other surface-exposed proteins from diverse bacteria are encoded in the genomes in two paralogous forms that differ by a ~45 amino acid fragment, which comprises a novel conserved domain. Sequence analysis of this domain revealed a conserved DxDxDGxxCE motif, which is strikingly similar to the Ca2+-binding loop of the calmodulin-like EF-hand domains, suggesting an evolutionary relationship between them. Functions of many of the other proteins in which the novel domain, named Excalibur (extracellular calcium-binding region), is found, as well as a structural model of its conserved motif are consistent with the notion that the Excalibur domain binds calcium. This domain is but one more example of the diversity of structural contexts surrounding the EF-hand-like calcium-binding loop in bacteria. This loop is thus more widespread than hitherto recognised and the evolution of EF-hand-like domains is probably more complex than previously appreciated.',
'FA58C' => 'SM00231____Coagulation factor 5/8 C-terminal domain, discoidin domain____Cell surface-attached carbohydrate-binding domain, present in eukaryotes and assumed to have horizontally transferred to eubacterial genomes.',
'FABD' => 'SM00808____F-actin binding domain (FABD) ____FABD is the F-actin binding domain of Bcr-Abl and its cellular counterpart c-Abl. The Bcr-Abl tyrosine kinase causes different forms of leukemia in humans. Depending on its position within the cell, Bcr-Abl differentially affects cellular growth. The FABD forms a compact left-handed four-helix bundle in solution. ',
'FAS1' => 'SM00554____Four repeated domains in the Fasciclin I family of proteins, present in many other contexts.____',
'FBD' => 'SM00579____domain in FBox and BRCT domain containing plant proteins____',
'FBG' => 'SM00186____Fibrinogen-related domains (FReDs)____Domain present at the C-termini of fibrinogen beta and gamma chains, and a variety of fibrinogen-related proteins, including tenascin and  Drosophila scabrous.',
'FBOX' => 'SM00256____A Receptor for Ubiquitination Targets____',
'FCD' => 'SM00895________This entry represents the C-terminal ligand binding domain of many members of the GntR family. This domain probably binds to a range of effector molecules that regulate the transcription of genes through the action of the N-terminal DNA-binding domain. This domain is found in and that are regulators of sugar biosynthesis operons. Many bacterial transcription regulation proteins bind DNA through a helix-turn-helix (HTH) motif, which can be classified into subfamilies on the basis of sequence similarities. The HTH GntR family has many members distributed among diverse bacterial groups that regulate various biological processes. It was named GntR after the Bacillus subtilis repressor of the gluconate operon. In general, these proteins contain a DNA-binding HTH domain at the N terminus, and an effector binding or oligomerisation domain at the C terminus. The winged-helix DNA-binding domain is well conserved in structure for the whole of the GntR family, and is similar in structure to other transcriptional regulator families. The C-terminal effector-binding and oligomerisation domains are more variable and are consequently used to define the subfamilies. Based on the sequence and structure of the C-terminal domains, the GtnR family can be divided into four major groups, as represented by FadR, HutC, MocR and YtrA, as well as some minor groups such as those represented by AraR and PlmA.',
'FCH' => 'SM00055____Fes/CIP4 homology domain____Alignment extended from original report. Highly alpha-helical. Also known as the RAEYL motif or the S. pombe Cdc15 N-terminal domain.',
'FDX-ACB' => 'SM00896____Ferredoxin-fold anticodon binding domain____This is the anticodon binding domain found in some phenylalanyl tRNA synthetases. The domain has a ferredoxin fold, consisting of an alpha+beta sandwich with anti-parallel beta-sheets (beta-alpha-beta x2).',
'FES' => 'SM00525________iron-sulpphur binding domain in DNA-(apurinic or apyrimidinic site) lyase (subfamily of ENDO3)',
'FF' => 'SM00441____Contains two conserved F residues____A novel motif that often accompanies WW domains. Often contains two conserved Phe (F) residues. ',
'FGF' => 'SM00442____Acidic and basic fibroblast growth factor family.____Mitogens that stimulate growth or differentiation of cells of mesodermal or neuroectodermal origin. The family play essential roles in patterning and differentiation during vertebrate embryogenesis, and have neurotrophic activities.',
'FH' => 'SM00339____FORKHEAD____FORKHEAD, also known as a "winged helix"',
'FH2' => 'SM00498____Formin Homology 2 Domain____FH proteins control rearrangements of the actin cytoskeleton, especially in the context of cytokinesis and cell polarisation. Members of this family have been found to interact with Rho-GTPases, profilin and other actin-assoziated proteins. These interactions are mediated by the proline-rich FH1 domain, usually located in front of FH2 (but not listed in SMART). Despite this cytosolic function, vertebrate formins have been assigned functions within the nucleus. A set of Formin-Binding Proteins (FBPs) has been shown to bind FH1 with their WW domain.',
'FHA' => 'SM00240____Forkhead associated domain____Found in eukaryotic and prokaryotic proteins. Putative nuclear signalling domain.',
'FIMAC' => 'SM00057____factor I membrane attack complex____',
'FIST' => 'SM00897____FIST N domain____The FIST N domain is a novel sensory domain, which is present in signal transduction proteins from Bacteria, Archaea and Eukarya. Chromosomal proximity of FIST-encoding genes to those coding for proteins involved in amino acid metabolism and transport suggest that FIST domains bind small ligands, such as amino acids.',
'FMN_bind' => 'SM00900________This conserved region includes the FMN-binding site of the NqrC protein as well as the NosR and NirI regulatory proteins. ',
'FN1' => 'SM00058____Fibronectin type 1 domain____One of three types of internal repeat within the plasma protein, fibronectin. Found also in coagulation factor XII, HGF activator and tissue-type plasminogen activator. In t-PA and fibronectin, this domain type  contributes to fibrin-binding.',
'FN2' => 'SM00059____Fibronectin type 2 domain____One of three types of internal repeat within the plasma protein, fibronectin. Also occurs in coagulation factor XII, 2 type IV collagenases, PDC-109, and cation-independent mannose-6-phosphate and secretory phospholipase A2  receptors. In fibronectin, PDC-109, and the collagenases, this domain contributes to  collagen-binding function.',
'FN3' => 'SM00060____Fibronectin type 3 domain____One of three types of internal repeat within the plasma protein, fibronectin. The tenth fibronectin type III repeat contains a RGD cell recognition  sequence in a flexible loop between 2 strands. Type III modules are present in both extracellular and intracellular proteins.',
'FOLN' => 'SM00274____Follistatin-N-terminal domain-like____Follistatin-N-terminal domain-like, EGF-like. Region distinct from the kazal-like sequence',
'FRG' => 'SM00901________This domain contains a conserved N-terminal (F/Y)RG motif. It is functionally uncharacterised. ',
'FRI' => 'SM00063____Frizzled____Drosophila melanogaster frizzled mediates signalling that polarises a precursor cell along the anteroposterior axis. Homologues of the  N-terminal region of frizzled exist either as transmembrane or secreted molecules. Frizzled homologues are reported to be receptors for the Wnt growth factors. (Not yet in MEDLINE: the FRI domain occurs in several receptor tyrosine kinases [Xu, Y.K. and Nusse, Curr. Biol. 8 R405-R406 (1998); Masiakowski, P. and Yanopoulos, G.D., Curr. Biol. 8, R407 (1998)].',
'FTP' => 'SM00607____eel-Fucolectin Tachylectin-4 Pentaxrin-1 Domain____',
'FU' => 'SM00261____Furin-like repeats____',
'FYRC' => 'SM00542____"FY-rich" domain, C-terminal region____is sometimes closely juxtaposed with the N-terminal region (FYRN), but sometimes is far distant. Unknown function, but occurs frequently in chromatin-associated proteins.',
'FYRN' => 'SM00541____"FY-rich" domain, N-terminal region____is sometimes closely juxtaposed with the C-terminal region (FYRC), but sometimes is far distant. Unknown function, but occurs frequently in chromatin-associated proteins.',
'FYVE' => 'SM00064____Protein present in Fab1, YOTB, Vac1, and EEA1____The FYVE zinc finger is named after four proteins where it was first found: Fab1, YOTB/ZK632.12, Vac1, and EEA1. The FYVE finger has been shown to bind two Zn2+ ions. The FYVE finger has eight potential zinc coordinating cysteine positions. The FYVE finger is structurally related to the <a href="/smart/do_annotation.pl?BLAST=DUMMY&DOMAIN=249">PHD finger</a> and the <a href="/smart/do_annotation.pl?BLAST=DUMMY&DOMAIN=184">RING finger</a>. Many members of this family also include two histidines in a motif R+HHC+XCG, where + represents a charged residue and X any residue.  The FYVE finger functions in the membrane recruitment of cytosolic proteins by binding to phosphatidylinositol 3-phosphate (PI3P), which is prominent on endosomes. The R+HHC+XCG motif is critical for PI3P binding.',
'Fapy_DNA_glyco' => 'SM00898____Formamidopyrimidine-DNA glycosylase N-terminal domain____This entry represents the catalytic domain of DNA glycosylase/AP lyase enzymes, which are involved in base excision repair of DNA damaged by oxidation or by mutagenic agents. Most damage to bases in DNA is repaired by the base excision repair pathway PUBMED:15588838. These enzymes are primarily from bacteria, and have both DNA glycosylase activity and AP lyase activity. Examples include formamidopyrimidine-DNA glycosylases (Fpg; MutM) and endonuclease VIII (Nei). Formamidopyrimidine-DNA glycosylases (Fpg, MutM) is a trifunctional DNA base excision repair enzyme that removes a wide range of oxidation-damaged bases (N-glycosylase activity; ) and cleaves both the 3\'- and 5\'-phosphodiester bonds of the resulting apurinic/apyrimidinic site (AP lyase activity; ). Fpg has a preference for oxidised purines, excising oxidized purine bases such as 7,8-dihydro-8-oxoguanine (8-oxoG). ITs AP (apurinic/apyrimidinic) lyase activity introduces nicks in the DNA strand, cleaving the DNA backbone by beta-delta elimination to generate a single-strand break at the site of the removed base with both 3\'- and 5\'-phosphates. Fpg is a monomer composed of 2 domains connected by a flexible hinge PUBMED:10921868. The two DNA-binding motifs (a zinc finger and the helix-two-turns-helix motifs) suggest that the oxidized base is flipped out from double-stranded DNA in the binding mode and excised by a catalytic mechanism similar to that of bifunctional base excision repair enzymes PUBMED:10921868. Fpg binds one ion of zinc at the C-terminus, which contains four conserved and essential cysteines PUBMED:8473347, PUBMED:7704272. Endonuclease VIII (Nei) has the same enzyme activities as Fpg above, but with a preference for oxidized pyrimidines, such as thymine glycol, 5,6-dihydrouracil and 5,6-dihydrothymine PUBMED:15232006. These protein contains three structural domains: an N-terminal catalytic core domain, a central helix-two turn-helix (H2TH) module and a C-terminal zinc finger (see PDB:1K82) PUBMED:11912217. The N-terminal catalytic domain and the C-terminal zinc finger straddle the DNA with the long axis of the protein oriented roughly orthogonal to the helical axis of the DNA. Residues that contact DNA are located in the catalytic domain and in a beta-hairpin loop formed by the zinc finger PUBMED:12055620.',
'Fe_hyd_SSU' => 'SM00902____Iron hydrogenase small subunit____Many microorganisms, such as methanogenic, acetogenic, nitrogen-fixing, photosynthetic, or sulphate-reducing bacteria, metabolise hydrogen. Hydrogen activation is mediated by a family of enzymes, termed hydrogenases, which either provide these organisms with reducing power from hydrogen oxidation, or act as electron sinks. There are two hydrogenases families that differ functionally from each other: NiFe hydrogenases tend to be more involved in hydrogen oxidation, while Iron-only FeFe (Fe only) hydrogenases in hydrogen production. Fe only hydrogenases show a common core structure, which contains a moiety, deeply buried inside the protein, with an Fe-Fe dinuclear centre, nonproteic bridging, terminal CO and CN- ligands attached to each of the iron atoms, and a dithio moiety, which also bridges the two iron atoms and has been tentatively assigned as a di(thiomethyl)amine. This common core also harbours three [4Fe-4S] iron-sulphur clusters PUBMED:11921392. In FeFe hydrogenases, as in NiFe hydrogenases, the set of iron-sulphur clusters is dispersed regularly between the dinuclear Fe-Fe centre and the molecular surface. These clusters are distant by about 1.2 nm from each other but the [4Fe-4S] cluster closest to the dinuclear centre is covalently bound to one of the iron atoms though a thiolate bridging ligand. The moiety including the dinuclear centre, the thiolate bridging ligand, and the proximal [4Fe-4S] cluster is known as the H-cluster. A channel, lined with hydrophobic amino acid side chains, nearly connects the dinuclear centre and the molecular surface. Furthermore hydrogen-bonded water molecule sites have been identified at the interior and at the surface of the protein. The small subunit is comprised of alternating random coil and alpha helical structures that encompass the large subunit in a novel protein fold PUBMED:10368269.',
'FeoA' => 'SM00899________This entry represents the core domain of the ferrous iron (Fe2+) transport protein FeoA found in bacteria. This domain also occurs at the C-terminus in related proteins. The transporter Feo is composed of three proteins: FeoA a small, soluble SH3-domain protein probably located in the cytosol; FeoB, a large protein with a cytosolic N-terminal G-protein domain and a C-terminal integral inner-membrane domain containing two \'Gate\' motifs which likely functions as the Fe2+ permease; and FeoC, a small protein apparently functioning as an [Fe-S]-dependent transcriptional repressor. Feo allows the bacterial cell to acquire iron from its environment. ',
'Flavin_Reduct' => 'SM00903____Flavin reductase like domain____This entry represents the FMN-binding domain found in NAD(P)H-flavin oxidoreductases (flavin reductases), a class of enzymes capable of producing reduced flavin for bacterial bioluminescence and other biological processes. This domain is also found in various other oxidoreductase and monooxygenase enzymes PUBMED:12829278, PUBMED:15461461, PUBMED:11017201. This domain consists of a beta-barrel with Greek key topology, and is related to the ferredoxin reductase-like FAD-binding domain. The flavin reductases have a different dimerisation mode than that found in the PNP oxidase-like family, which also carries an FMN-binding domain with a similar topology.',
'Flavokinase' => 'SM00904____Riboflavin kinase____Riboflavin is converted into catalytically active cofactors (FAD and FMN) by the actions of riboflavin kinase, which converts it into FMN, and FAD synthetase, which adenylates FMN to FAD. Eukaryotes usually have two separate enzymes, while most prokaryotes have a single bifunctional protein that can carry out both catalyses, although exceptions occur in both cases. While eukaryotic monofunctional riboflavin kinase is orthologous to the bifunctional prokaryotic enzyme PUBMED:14580199, the monofunctional FAD synthetase differs from its prokaryotic counterpart, and is instead related to the PAPS-reductase family PUBMED:17049878. The bacterial FAD synthetase that is part of the bifunctional enzyme has remote similarity to nucleotidyl transferases and, hence, it may be involved in the adenylylation reaction of FAD synthetases PUBMED:12517446. This entry represents riboflavin kinase, which occurs as part of a bifunctional enzyme or a stand-alone enzyme.',
'Flu_M1_C' => 'SM00759____Influenza Matrix protein (M1) C-terminal domain____This region is thought to be a second domain of the M1 matrix protein.',
'FolB' => 'SM00905____Dihydroneopterin aldolase____Dihydroneopterin aldolase catalyses the conversion of 7,8-dihydroneopterin to 6-hydroxymethyl-7,8-dihydropterin in the biosynthetic pathway of tetrahydrofolate. In the opportunistic pathogen Pneumocystis carinii, dihydroneopterin aldolase function is expressed as the N-terminal portion of the multifunctional folic acid synthesis protein (Fas). This region encompasses two domains, FasA and FasB, which are 27% amino acid identical. FasA and FasB also share significant amino acid sequence similarity with bacterial dihydroneopterin aldolases. This region consists of two tandem sequences each homologous to folB and which form tetramers PUBMED:9709001.',
'FtsA' => 'SM00842____Cell division protein FtsA____FtsA is essential for bacterial cell division, and co-localizes to the septal ring with FtsZ. It has been suggested that the interaction of FtsA-FtsZ has arisen through coevolution in different bacterial strains PUBMED:9352931. ',
'Ftsk_gamma' => 'SM00843________This domain directs oriented DNA translocation and forms a winged helix structure. Mutated proteins with substitutions in the FtsK gamma DNA-recognition helix are impaired in DNA binding. ',
'Fungal_trans' => 'SM00906____Fungal specific transcription factor domain____This domain is found in a number of fungal transcription factors including transcriptional activator xlnR, yeast regulatory protein GAL4, and other transcription proteins regulating a variety of cellular and metabolic processes.',
'G2F' => 'SM00682____G2 nidogen domain and fibulin____',
'GA' => 'SM00844____GA module____The protein G-related albumin-binding (GA) module is composed of three alpha helices PUBMED:9086265. This module is found in a range of bacterial cell surface proteins. The GA module from the Peptostreptococcus magnus albumin-binding protein (PAB) shows a strong affinity for albumin.',
'GAF' => 'SM00065____Domain present in phytochromes and cGMP-specific phosphodiesterases.____Mutations within these domains in PDE6B result in autosomal recessive  inheritance of retinitis pigmentosa. ',
'GAL4' => 'SM00066____GAL4-like Zn(II)2Cys6 (or C6 zinc) binuclear cluster DNA-binding domain____Gal4 is a positive regulator for the gene expression of the galactose- induced genes of S. cerevisiae. Is present only in fungi.',
'GAS2' => 'SM00243____Growth-Arrest-Specific Protein 2 Domain____GROWTH-ARREST-SPECIFIC PROTEIN 2 Domain',
'GASTRIN' => 'SM00029____gastrin / cholecystokinin / caerulein family____This family gathers small proteins of about 100 130 amino acids that act as hormones, among them gastrin, cholecystokinin and preprocaerulein which stimulate gastric, biliary, and pancreatic secretion and smooth muscle contraction. ',
'GDNF' => 'SM00907____GDNF/GAS1 domain____This cysteine rich domain is found in multiple copies in GNDF and GAS1 proteins. GDNF and neurturin (NTN) receptors are potent survival factors for sympathetic, sensory and central nervous system neurons PUBMED:16551639, PUBMED:9192899. GDNF and neurturin promote neuronal survival by signaling through similar multicomponent receptors that consist of a common receptor tyrosine kinase and a member of a GPI-linked family of receptors that determines ligand specificity PUBMED:9192898. ',
'GED' => 'SM00302____Dynamin GTPase effector domain____',
'GEL' => 'SM00262____Gelsolin homology domain____Gelsolin/severin/villin homology domain. Calcium-binding and actin-binding. Both intra- and extracellular domains.',
'GGDEF' => 'SM00267____diguanylate cyclase____Diguanylate cyclase, present in a variety of bacteria.',
'GGL' => 'SM00224____G protein gamma subunit-like motifs____',
'GHA' => 'SM00067____Glycoprotein hormone alpha chain homologues.____Also called gonadotropins. Glycoprotein hormones consist of two glycosylated chains (alpha and beta) of similar topology.',
'GHB' => 'SM00068____Glycoprotein hormone beta chain homologues.____Also called gonadotropins. Glycoprotein hormones consist of two glycosylated chains (alpha and beta) of similar topology.',
'GIT' => 'SM00555____Helical motif in the GIT family of ADP-ribosylation factor GTPase-activating proteins____Helical motif in the GIT family of ADP-ribosylation factor GTPase-activating proteins, and in yeast Spa2p and Sph1p (CPP; unpublished results). In p95-APP1 the N-terminal GIT motif might be involved in binding PIX.',
'GIYc' => 'SM00465____GIY-YIG type nucleases (URI domain)____',
'GLA' => 'SM00069____Domain containing Gla (gamma-carboxyglutamate) residues.____A hyaluronan-binding domain found in proteins associated with the extracellular matrix, cell adhesion and cell migration.',
'GLECT' => 'SM00276____Galectin____Galectin - galactose-binding lectin',
'GLUCA' => 'SM00070____Glucagon like hormones____',
'GPS' => 'SM00303____G-protein-coupled receptor proteolytic site domain____Present in latrophilin/CL-1, sea urchin REJ and polycystin.',
'GRAM' => 'SM00568____domain in glucosyltransferases, myotubularins and other putative membrane-associated proteins____',
'GRAN' => 'SM00277____Granulin____',
'GS' => 'SM00467____GS motif____Aa approx. 30 amino acid motif that precedes the kinase domain in types I and II TGF beta receptors. Mutation of two or more of the serines or threonines in the TTSGSGSG of TGF-beta type I receptor impairs phosphorylation and signaling activity.',
'GYF' => 'SM00444____Contains conserved Gly-Tyr-Phe residues____Proline-binding domain in CD2-binding protein. Contains conserved Gly-Tyr-Phe residues. ',
'GYR' => 'SM00713____Motif of unknown function with conserved Gly, Tyr, Arg tripeptide in Drosophila proteins.____',
'G_alpha' => 'SM00275____G protein alpha subunit____Subunit of G proteins that contains the guanine nucleotide binding site',
'G_patch' => 'SM00443____glycine rich nucleic binding domain____A predicted glycine rich nucleic binding domain found in the splicing factor 45, SON DNA binding protein and D-type Retrovirus- polyproteins.',
'Gal-bind_lectin' => 'SM00908____Galactoside-binding lectin____Animal lectins display a wide variety of architectures. They are classified according to the carbohydrate-recognition domain (CRD) of which there are two main types, S-type and C-type. Galectins (previously S-lectins) bind exclusively beta-galactosides like lactose. They do not require metal ions for activity. Galectins are found predominantly, but not exclusively in mammals PUBMED:8124704. Their function is unclear. They are developmentally regulated and may be involved in differentiation, cellular regulation and tissue construction.',
'Galanin' => 'SM00071____Galanin____Galanin [1,2,3] is a neuropeptide that controls various biological activities: it regulates the release growth hormone, inhibits the release of insulin and somatostatin, contracts smooth muscle of the gastrointestinal and genitourinary tract and may be involved in the control of adrenal secretion',
'GatB_Yqey' => 'SM00845____GatB domain____This domain is found in GatB and proteins related to bacterial Yqey. It is about 140 amino acid residues long. This domain is found at the C terminus of GatB which transamidates Glu-tRNA to Gln-tRNA. The function of this domain is uncertain. It does however suggest that Yqey and its relatives have a role in tRNA metabolism.',
'Germane' => 'SM00909____Sporulation and spore germination____The GerMN domain is a region of approximately 100 residues that is found, duplicated, in the Bacillus GerM protein and is implicated in both sporulation and spore germination. The domain is found in a number of different bacterial species both alone and in association with other domains such as Amidase_3 PF01520 Gmad1 and Gmad2. It is predicted to have a novel alpha-beta fold.',
'Glyco_10' => 'SM00633____Glycosyl hydrolase family 10____',
'Glyco_18' => 'SM00636________',
'Glyco_25' => 'SM00641____Glycosyl hydrolases family 25____',
'Glyco_32' => 'SM00640____Glycosyl hydrolases family 32____',
'GoLoco' => 'SM00390____LGN motif, putative GEFs specific for G-alpha GTPases____GEF specific for Galpha_i proteins',
'Gp_dh_N' => 'SM00846____Glyceraldehyde 3-phosphate dehydrogenase, NAD binding domain____GAPDH is a tetrameric NAD-binding enzyme involved in glycolysis and glyconeogenesis. N-terminal domain is a Rossmann NAD(P) binding fold.',
'Grip' => 'SM00755____golgin-97, RanBP2alpha,Imh1p and p230/golgin-245____',
'GuKc' => 'SM00072____Guanylate kinase homologues.____Active enzymes catalyze ATP-dependent phosphorylation of GMP to GDP. Structure resembles that of adenylate kinase. So-called membrane-associated guanylate kinase homologues (MAGUKs) do not possess guanylate kinase activities; instead at least some possess protein-binding functions.',
'H15' => 'SM00526____Domain in histone families 1 and 5____',
'H2A' => 'SM00414____Histone 2A____',
'H2B' => 'SM00427____Histone H2B____',
'H3' => 'SM00428____Histone H3____',
'H4' => 'SM00417____Histone H4____',
'HA2' => 'SM00847____Helicase associated domain (HA2)  Add an annotation____This presumed domain is about 90 amino acid residues in length. It is found is a diverse set of RNA helicases. Its function is unknown, however it seems likely to be involved in nucleic acid binding.',
'HALZ' => 'SM00340____homeobox associated leucin zipper____',
'HAMP' => 'SM00304____HAMP (Histidine kinases, Adenylyl cyclases, Methyl binding proteins, Phosphatases) domain____',
'HAT' => 'SM00386____HAT (Half-A-TPR) repeats____Present in several RNA-binding proteins. Structurally and sequentially thought to be similar to TPRs.',
'HATPase_c' => 'SM00387____Histidine kinase-like ATPases____Histidine kinase-, DNA gyrase B-, phytochrome-like ATPases.',
'HDAC_interact' => 'SM00761____Histone deacetylase (HDAC) interacting____This domain is found on transcriptional regulators. It forms interactions with histone deacetylases.',
'HDc' => 'SM00471____Metal dependent phosphohydrolases with conserved \'HD\' motif.____Includes eukaryotic cyclic nucleotide phosphodiesterases (PDEc). This profile/HMM does not detect HD homologues in bacterial glycine aminoacyl-tRNA synthetases (beta subunit).',
'HECTc' => 'SM00119____Domain Homologous to E6-AP Carboxyl Terminus with ____E3 ubiquitin-protein ligases. Can bind to E2 enzymes.',
'HELICc' => 'SM00490____helicase superfamily c-terminal domain____',
'HELICc2' => 'SM00491________',
'HELICc3' => 'SM00492________',
'HEPN' => 'SM00748____Higher Eukarytoes and Prokaryotes Nucleotide-binding domain____',
'HIRAN' => 'SM00910________The HIRAN protein (HIP116, Rad5p N-terminal) is found in the N-terminal regions of the SWI2/SNF2 proteins typified by HIP116 and Rad5p. HIRAN is found as a standalone protein in several bacteria and prophages, or fused to other catalytic domains, such as a nuclease of the restriction endonuclease fold and TDP1-like DNA phosphoesterases, in the eukaryotes PUBMED:16627993. It has been predicted that this protein functions as a DNA-binding domain that probably recognises features associated with damaged DNA or stalled replication forks PUBMED:16627993',
'HLH' => 'SM00353____helix loop helix domain____',
'HMG' => 'SM00398____high mobility group____',
'HMG17' => 'SM00527____domain in high mobilty group proteins HMG14 and HMG 17____',
'HNHc' => 'SM00507____HNH nucleases____',
'HNS' => 'SM00528____Domain in histone-like proteins of HNS family____',
'HOLI' => 'SM00430____Ligand binding domain of hormone receptors____',
'HOX' => 'SM00389____Homeodomain____DNA-binding factors that are involved in the transcriptional regulation of key developmental processes',
'HPT' => 'SM00073____Histidine Phosphotransfer domain____Contains an active histidine residue that mediates phosphotransfer reactions. Domain detected only in eubacteria. This alignment is an extension to that shown in the Cell structure paper.',
'HRDC' => 'SM00341____Helicase and RNase D C-terminal____Hypothetical role in nucleic acid binding. Mutations in the HRDC domain cause human disease.',
'HSA' => 'SM00573____domain in helicases and associated with SANT domains____',
'HSF' => 'SM00415____heat shock factor____',
'HTH_ARAC' => 'SM00342____helix_turn_helix, arabinose operon control protein____',
'HTH_ARSR' => 'SM00418____helix_turn_helix, Arsenical Resistance Operon Repressor____',
'HTH_ASNC' => 'SM00344____helix_turn_helix ASNC type____AsnC: an autogenously regulated activator of asparagine synthetase A transcription in Escherichia coli) ',
'HTH_CRP' => 'SM00419____helix_turn_helix, cAMP Regulatory protein____',
'HTH_DEOR' => 'SM00420____helix_turn_helix, Deoxyribose operon repressor____',
'HTH_DTXR' => 'SM00529____Helix-turn-helix diphteria tox regulatory element____iron dependent repressor',
'HTH_GNTR' => 'SM00345____helix_turn_helix gluconate operon transcriptional repressor____',
'HTH_ICLR' => 'SM00346____helix_turn_helix isocitrate lyase regulation____',
'HTH_LACI' => 'SM00354____helix_turn _helix lactose operon repressor____',
'HTH_LUXR' => 'SM00421____helix_turn_helix, Lux Regulon____lux regulon (activates the bioluminescence operon',
'HTH_MARR' => 'SM00347____helix_turn_helix multiple antibiotic resistance protein____',
'HTH_MERR' => 'SM00422____helix_turn_helix, mercury resistance____',
'HTH_XRE' => 'SM00530____Helix-turn-helix XRE-family like proteins ____',
'HTTM' => 'SM00752____Horizontally Transferred TransMembrane Domain____Sequence analysis of vitamin K dependent gamma-carboxylases (VKGC) revealed the presence of a novel domain, HTTM (Horizontally Transferred TransMembrane) in its N-terminus. In contrast to most known domains, HTTM contains four transmembrane regions. Its occurrence in eukaryotes, bacteria and archaea is more likely caused by horizontal gene transfer than by early invention. The conservation of VKGC catalytic sites indicates an enzymatic function also for the other family members. ',
'HWE_HK' => 'SM00911____HWE histidine kinase____The HWE domain is found in a subset of two-component system kinases, belonging to the same superfamily as PUBMED:14702314. In PUBMED:14702314, the HWE family was defined by the presence of conserved a H residue and a WXE motifs and was limited to members of the proteobacteria. However, many homologues of this domain are lack the WXE motif. Furthermore, homologues are found in a wide range of Gram-positive and Gram-negative bacteria as well as in several archaea.',
'HX' => 'SM00120____Hemopexin-like repeats.____Hemopexin is a heme-binding protein that transports heme to the liver. Hemopexin-like repeats occur in vitronectin and some matrix  metalloproteinases family (matrixins). The HX repeats of some matrixins  bind tissue inhibitor of metalloproteinases (TIMPs).',
'HYDRO' => 'SM00075____Hydrophobins____',
'Haemagg_act' => 'SM00912____haemagglutination activity domain____This domain is suggested to be a carbohydrate- dependent haemagglutination activity site PUBMED:11703654. It is found in a range of haemagglutinins and haemolysins.',
'HhH1' => 'SM00278____Helix-hairpin-helix DNA-binding motif class 1____',
'HhH2' => 'SM00279____Helix-hairpin-helix class 2 (Pol1 family) motifs____',
'HintC' => 'SM00305____Hint (Hedgehog/Intein) domain C-terminal region____Hedgehog/Intein domain, C-terminal region. Domain has been split to accommodate large insertions of endonucleases.',
'HintN' => 'SM00306____Hint (Hedgehog/Intein) domain N-terminal region____Hedgehog/Intein domain, N-terminal region. Domain has been split to accommodate large insertions of endonucleases.',
'HisKA' => 'SM00388____His Kinase A (phosphoacceptor) domain____Dimerisation and phosphoacceptor domain of histidine kinases. ',
'HormR' => 'SM00008____Domain present in hormone receptors____',
'Hr1' => 'SM00742____Rho effector or protein kinase C-related kinase homology region 1 homologues____Alpha-helical domain found in vertebrate PRK1 and yeast PKC1 protein kinases C. The HR1 in rhophilin bind RhoGTP; those in PRK1 bind RhoA and RhoB. Also called RBD - Rho-binding domain ',
'IB' => 'SM00121____Insulin growth factor-binding protein homologues____High affinity binding partners of insulin-like growth factors.',
'IBN_N' => 'SM00913____Importin-beta N-terminal domain____Members of the importin-beta (karyopherin-beta) family can bind and transport cargo by themselves, or can form heterodimers with importin-alpha. As part of a heterodimer, importin-beta mediates interactions with the pore complex, while importin-alpha acts as an adaptor protein to bind the nuclear localisation signal (NLS) on the cargo through the classical NLS import of proteins. Importin-beta is a helicoidal molecule constructed from 19 HEAT repeats. Many nuclear pore proteins contain FG sequence repeats that can bind to HEAT repeats within importins PUBMED:12372823, PUBMED:17161424, which is important for importin-beta mediated transport.',
'IBR' => 'SM00647____In Between Ring fingers____the domains occurs between pairs og RING fingers',
'IDEAL' => 'SM00914________This is a short protein of unknown function it is found at the C-terminus of proteins in the UPF0302 family. It is named after the sequence of the most conserved region in some members.',
'IENR1' => 'SM00497____Intron encoded nuclease repeat motif____Repeat of unknown function, but possibly DNA-binding via helix-turn-helix motif (Ponting, unpublished).',
'IENR2' => 'SM00496____Intron-encoded nuclease repeat 2____Short helical motif of unknown function (unpublished results).',
'IFabd' => 'SM00076____Interferon alpha, beta and delta.____Interferons produce antiviral and antiproliferative responses in cells. They are classified into five groups, all of them related but gamma-interferon. ',
'IG' => 'SM00409____Immunoglobulin____',
'IG_FLMN' => 'SM00557____Filamin-type immunoglobulin domains____These form a rod-like structure in the actin-binding cytoskeleton protein, filamin. The C-terminal repeats of filamin bind beta1-integrin (CD29).',
'IG_like' => 'SM00410____Immunoglobulin like____IG domains that cannot be classified into one of IGv1, IGc1, IGc2, IG.',
'IGc1' => 'SM00407____Immunoglobulin C-Type____',
'IGc2' => 'SM00408____Immunoglobulin C-2 Type____',
'IGv' => 'SM00406____Immunoglobulin V-Type____',
'IL1' => 'SM00125____Interleukin-1 homologues ____Cytokines with various biological functions. Interluekin 1 alpha and beta are also known as hematopoietin and catabolin.',
'IL10' => 'SM00188____Interleukin-10 family____Interleukin-10 inhibits the synthesis of a number of cytokines, including IFN-gamma, IL-2, IL-3, TNF and GM-CSF produced by activated macrophages and by helper T cells. ',
'IL2' => 'SM00189____Interleukin-2 family____Interleukin-2 is a cytokine produced by T-helper cells in response to antigenic or mitogenic stimulation. This protein is required for T-cell proliferation and other activities crucial to the regulation of the immune response. ',
'IL4_13' => 'SM00190____Interleukins 4 and 13____Interleukins-4 and -13 are cytokines involved in inflammatory and immune responses. IL-4 stimulates B and T cells. ',
'IL6' => 'SM00126____Interleukin-6 homologues____Family includes granulocyte colony-stimulating factor (G-CSF) and myelomonocytic growth factor (MGF). IL-6 is also known as B-cell stimulatory factor 2.',
'IL7' => 'SM00127____Interleukin-7 and interleukin-9 family.____IL-7 is a cytokine that acts as a growth factor for early lymphoid cells of both B- and T-cell lineages. IL-9 is a multifunctional cytokine that, although originally described as a T-cell growth factor, its function in T-cell response remains unclear.',
'ILWEQ' => 'SM00307____I/LWEQ domain____Thought to possess an F-actin binding function.',
'INB' => 'SM00187____Integrin beta subunits (N-terminal portion of extracellular region)____Portion of beta integrins that lies N-terminal to their EGF-like repeats. Integrins are cell adhesion molecules that mediate cell-extracellular  matrix and cell-cell interactions. They contain both alpha and beta subunits. Beta integrins are proposed to have a von Willebrand factor type-A "insert" or "I" -like domain (although this remains to be confirmed).',
'IPPc' => 'SM00128____Inositol polyphosphate phosphatase, catalytic domain homologues____Mg(2+)-dependent/Li(+)-sensitive enzymes.',
'IPT' => 'SM00429____ig-like, plexins, transcription factors____',
'IQ' => 'SM00015____Short calmodulin-binding motif containing conserved Ile and Gln residues.____Calmodulin-binding motif.',
'IRF' => 'SM00348____interferon regulatory factor____interferon regulatory factor, also known as trytophan pentad repeat',
'IRO' => 'SM00548____Motif in Iroquois-class homeodomain proteins (only). Unknown function.____',
'ITAM' => 'SM00077____Immunoreceptor tyrosine-based activation motif____Motif that may be dually phosphorylated on tyrosine that links antigen receptors to downstream signalling machinery. ',
'IlGF' => 'SM00078____Insulin / insulin-like growth factor / relaxin family.____Family of proteins including insulin, relaxin, and IGFs. Insulin decreases blood glucose concentration.',
'Inhibitor_I29' => 'SM00848____Cathepsin propeptide inhibitor domain (I29)____This domain is found at the N-terminus of some C1 peptidases such as Cathepsin L where it acts as a propeptide. There are also a number of proteins that are composed solely of multiple copies of this domain such as the peptidase inhibitor salarin. This family is classified as I29 by MEROPS. Peptide proteinase inhibitors can be found as single domain proteins or as single or multiple domains within proteins; these are referred to as either simple or compound inhibitors, respectively. In many cases they are synthesised as part of a larger precursor protein, either as a prepropeptide or as an N-terminal domain associated with an inactive peptidase or zymogen. This domain prevents access of the substrate to the active site. Removal of the N-terminal inhibitor domain either by interaction with a second peptidase or by autocatalytic cleavage activates the zymogen. Other inhibitors interact direct with proteinases using a simple noncovalent lock and key mechanism; while yet others use a conformational change-based trapping mechanism that depends on their structural and thermodynamic properties.',
'Int_alpha' => 'SM00191____Integrin alpha (beta-propellor repeats).____Integrins are cell adhesion molecules that mediate cell-extracellular  matrix and cell-cell interactions. They contain both alpha and beta subunits. Alpha integrins are proposed to contain a domain containing a 7-fold repeat that adopts a beta-propellor fold. Some of these domains contain an inserted von Willebrand factor type-A domain. Some repeats  contain putative calcium-binding sites. The 7-fold repeat domain is homologous to a similar domain in phosphatidylinositol-glycan-specific  phospholipase D.',
'JAB_MPN' => 'SM00232____JAB/MPN domain____Domain in Jun kinase activation domain binding protein and proteasomal subunits. Domain at Mpr1p and Pad1p N-termini. Domain of unknown function.',
'JHBP' => 'SM00700____Juvenile hormone binding protein domains in insects.____The juvenile hormone exerts pleiotropic functions during insect life cycles and its binding proteins regulate these functions.',
'Jacalin' => 'SM00915____Jacalin-like lectin domain____This entry represents a mannose-binding lectin domain with a beta-prism fold consisting of three 4-stranded beta-sheets, with an internal pseudo 3-fold symmetry. Some lectins in this group stimulate distinct T- and B- cell functions, such as Jacalin, which binds to the T-antigen and acts as an agglutinin. This domain is found in 1 to 6 copies in lectins. The domain is also found in the salt-stress induced protein from rice and an animal prostatic spermine-binding protein.',
'JmjC' => 'SM00558____A domain family that is part of the cupin metalloenzyme superfamily.____Probable enzymes, but of unknown functions, that regulate chromatin reorganisation processes (Clissold and Ponting, in press).',
'JmjN' => 'SM00545____Small domain found in the jumonji family of transcription factors____To date, this domain always co-occurs with the JmjC domain (although the reverse is not true). ',
'KAZAL' => 'SM00280____Kazal type serine protease inhibitors____Kazal type serine protease inhibitors and follistatin-like domains.',
'KH' => 'SM00322____K homology RNA-binding domain____',
'KIND' => 'SM00750____kinase non-catalytic C-lobe domain____It is an interaction domain identified as being similar to the C-terminal protein kinase catalytic fold (C lobe). Its presence at the N terminus of signalling proteins and the absence of the active-site residues in the catalytic and activation loops suggest that it folds independently and is likely to be non-catalytic. The occurrence of KIND only in metazoa implies that it has evolved from the catalytic protein kinase domain into an interaction domain possibly by keeping the substrate-binding features',
'KISc' => 'SM00129____Kinesin motor, catalytic domain. ATPase.____Microtubule-dependent molecular motors that play important roles in intracellular transport of organelles and in cell division. ',
'KOW' => 'SM00739____KOW (Kyprides, Ouzounis, Woese) motif.____Motif in ribosomal proteins, NusG, Spt5p, KIN17 and T54.',
'KR' => 'SM00130____Kringle domain____Named after a Danish pastry. Found in several serine proteases and in ROR-like receptors. Can occur in up to 38 copies (in apolipoprotein(a)). Plasminogen-like kringles possess affinity for free lysine and lysine- containing peptides. ',
'KRAB' => 'SM00349____krueppel associated box____',
'KU' => 'SM00131____BPTI/Kunitz family of serine protease inhibitors.____Serine protease inhibitors. One member of the family is encoded by an alternatively-spliced form of Alzheimer\'s amyloid beta-protein.',
'Kelch' => 'SM00612________',
'Knot1' => 'SM00505____Knottins____Knottins, representing plant lectins/antimicrobial peptides, plant proteinase/amylase inhibitors, plant gamma-thionins and arthropod defensins.',
'Ku78' => 'SM00559____Ku70 and Ku80 are 70kDa and 80kDa subunits of the Lupus Ku autoantigen____This is a single stranded DNA- and ATP-depedent helicase that has a role in chromosome translocation. This is a domain of unknown function C-terminal to its von Willebrand factor A domain, that also occurs in bacterial hypothetical proteins.',
'L27' => 'SM00569____domain in receptor targeting proteins Lin-2 and Lin-7____',
'L51_S25_CI-B8' => 'SM00916____Mitochondrial ribosomal protein L51 / S25 / CI-B8 domain____Proteins containing this domain are located in the mitochondrion and include ribosomal protein L51, and S25. This domain is also found in mitochondrial NADH-ubiquinone oxidoreductase B8 subunit (CI-B8) . It is not known whether all members of this family form part of the NADH-ubiquinone oxidoreductase and whether they are also all ribosomal proteins.',
'LA' => 'SM00715____Domain in the RNA-binding Lupus La protein; unknown function____',
'LCCL' => 'SM00603________',
'LDLa' => 'SM00192____Low-density lipoprotein receptor domain class A____Cysteine-rich repeat in the low-density lipoprotein (LDL) receptor that  plays a central role in mammalian cholesterol metabolism. The N-terminal  type A repeats in LDL receptor bind the lipoproteins. Other homologous  domains occur in related receptors, including the very low-density  lipoprotein receptor and the LDL receptor-related protein/alpha  2-macroglobulin receptor, and in proteins which are functionally  unrelated, such as the C9 component of complement. Mutations in the LDL receptor gene cause familial hypercholesterolemia.',
'LEM' => 'SM00540____in nuclear membrane-associated proteins____LEM, domain in nuclear membrane-associated proteins, including lamino-associated polypeptide 2 and emerin.',
'LH2' => 'SM00308____Lipoxygenase homology 2 (beta barrel) domain____',
'LIF_OSM' => 'SM00080____leukemia inhibitory factor ____OSM, Oncostatin M ',
'LIGANc' => 'SM00532____Ligase N family____',
'LIM' => 'SM00132____Zinc-binding domain present in Lin-11, Isl-1, Mec-3.____Zinc-binding domain family. Some LIM domains bind protein partners via tyrosine-containing motifs. LIM domains are found in many key regulators of developmental pathways.',
'LINK' => 'SM00445____Link (Hyaluronan-binding)____',
'LITAF' => 'SM00714____Possible membrane-associated motif in LPS-induced tumor necrosis factor alpha factor (LITAF), also known as PIG7, and other animal proteins.____',
'LMWPc' => 'SM00226____Low molecular weight phosphatase family____',
'LNS2' => 'SM00775________This domain is found in Saccharomyces cerevisiae protein SMP2, proteins with an N-terminal lipin domain and phosphatidylinositol transfer proteins. SMP2 is involved in plasmid maintenance and respiration. Lipin proteins are involved in adipose tissue development and insulin resistance.',
'LON' => 'SM00464____Found in ATP-dependent protease La (LON)____N-terminal domain of the ATP-dependent protease La (LON), present also in other bacterial ORFs.',
'LPD_N' => 'SM00638____Lipoprotein N-terminal Domain____',
'LRR' => 'SM00370____Leucine-rich repeats, outliers____',
'LRRCT' => 'SM00082____Leucine rich repeat C-terminal domain____',
'LRRNT' => 'SM00013____Leucine rich repeat N-terminal domain____',
'LRR_BAC' => 'SM00364____Leucine-rich repeats, bacterial type____',
'LRR_CC' => 'SM00367____Leucine-rich repeat - CC (cysteine-containing) subfamily____',
'LRR_RI' => 'SM00368____Leucine rich repeat, ribonuclease inhibitor type____',
'LRR_SD22' => 'SM00365____Leucine-rich repeat, SDS22-like subfamily____',
'LRR_TYP' => 'SM00369____Leucine-rich repeats, typical (most populated) subfamily____',
'LRRcap' => 'SM00446____occurring C-terminal to leucine-rich repeats____A motif occurring C-terminal to leucine-rich repeats in "sds22-like" and "typical" LRR-containing proteins.',
'LU' => 'SM00134____Ly-6 antigen / uPA receptor -like domain____Three-fold repeated domain in urokinase-type plasminogen activator receptor; occurs singly in other GPI-linked cell-surface glycoproteins (Ly-6 family, CD59, thymocyte B cell antigen, Sgp-2). Topology of these domains is similar to that of snake venom neurotoxins.',
'LY' => 'SM00135____Low-density lipoprotein-receptor YWTD domain____Type "B" repeats in low-density lipoprotein (LDL) receptor that  plays a central role in mammalian cholesterol metabolism. Also present in a variety of molecules similar to gp300/megalin.',
'LYZ1' => 'SM00263____ Alpha-lactalbumin / lysozyme C____',
'LYZ2' => 'SM00047____Lysozyme subfamily 2____Eubacterial enzymes distantly related to eukaryotic lysozymes.',
'Lactamase_B' => 'SM00849____Metallo-beta-lactamase superfamily____Apart from the beta-lactamases a number of other proteins contain this domain PUBMED:7588620. These proteins include thiolesterases, members of the glyoxalase II family, that catalyse the hydrolysis of S-D-lactoyl-glutathione to form glutathione and D-lactic acid and a competence protein that is essential for natural transformation in Neisseria gonorrhoeae and could be a transporter involved in DNA uptake. Except for the competence protein these proteins bind two zinc ions per molecule as cofactor.',
'LamB' => 'SM00281____Laminin B domain____',
'LamG' => 'SM00282____Laminin G domain____',
'LamGL' => 'SM00560____LamG-like jellyroll fold domain____',
'LamNT' => 'SM00136____Laminin N-terminal domain (domain VI)____N-terminal domain of laminins and laminin-related protein such as Unc-6/ netrins.',
'LeuA_dimer' => 'SM00917____LeuA allosteric (dimerisation) domain____This is the C-terminal regulatory (R) domain of alpha-isopropylmalate synthase, which catalyses the first committed step in the leucine biosynthetic pathway PUBMED:15159544. This domain, is an internally duplicated structure with a novel fold PUBMED:15159544. It comprises two similar units that are arranged such that the two -helices pack together in the centre, crossing at an angle of 34 degrees, sandwiched between the two three-stranded, antiparallel beta-sheets. The overall domain is thus constructed as a beta-alpha-beta three-layer sandwich PUBMED:15159544. ',
'Lig_chan-Glu_bd' => 'SM00918____Ligated ion channel L-glutamate- and glycine-binding site____This region, sometimes called the S1 domain, is the luminal domain just upstream of the first, M1, transmembrane region of transmembrane ion-channel proteins, and it binds L-glutamate and glycine. It is found in association with Lig_chan.',
'LisH' => 'SM00667____Lissencephaly type-1-like homology motif____Alpha-helical motif present in Lis1, treacle, Nopp140, some katanin p60 subunits, muskelin, tonneau, LEUNIG and numerous WD40 repeat-containing proteins. It is suggested that LisH motifs contribute to the regulation of microtubule dynamics, either by mediating dimerisation, or else by binding cytoplasmic dynein heavy chain or microtubules directly.',
'LysM' => 'SM00257____Lysin motif____',
'LytTR' => 'SM00850____LytTr DNA-binding domain____This domain is found in a variety of bacterial transcriptional regulators. The domain binds to a specific DNA sequence pattern.',
'MA' => 'SM00283____Methyl-accepting chemotaxis-like domains (chemotaxis sensory transducer).____Thought to undergo reversible methylation in response to attractants or repellants during bacterial chemotaxis.',
'MA3' => 'SM00544____Domain in DAP-5, eIF4G, MA-3 and other proteins.____Highly alpha-helical. May contain repeats and/or regions similar to MIF4G domains Ponting (TIBS) "Novel eIF4G domain homologues" in press',
'MACPF' => 'SM00457____membrane-attack complex / perforin____',
'MADF' => 'SM00595________subfamily of SANT domain',
'MADS' => 'SM00432________',
'MAM' => 'SM00137____Domain in meprin, A5, receptor protein tyrosine phosphatase mu (and others)____Likely to have an  adhesive  function. Mutations in the meprin MAM domain affect noncovalent associations within meprin oligomers. In receptor tyrosine phosphatase mu-like molecules the MAM domain is important for homophilic cell-cell interactions.',
'MANEC' => 'SM00765________The MANEC domain was formerly called MANSC. This domain, comprising 8 conserved cysteines, is found in the N terminus of higher multicellular animal membrane and extracellular proteins. It is postulated that this domain may play a role in the formation of protein complexes involving various protease activators and inhibitors. It is possible that some of the cysteine residues in the MANSC domain form structurally important disulfide bridges. All of the MANSC-containing proteins contain predicted transmembrane regions and signal peptides. It has been proposed that the MANSC domain in HAI-1 might function through binding with hepatocyte growth factor activator and matriptase. ',
'MATH' => 'SM00061____meprin and TRAF homology____',
'MBD' => 'SM00391____Methyl-CpG binding domain____Methyl-CpG binding domain, also known as the TAM (TTF-IIP5, ARBP, MeCP1) domain ',
'MBT' => 'SM00561____Present in Drosophila Scm, l(3)mbt, and vertebrate SCML2____Present in Drosophila Scm, l(3)mbt, and vertebrate SCML2. These proteins are involved in transcriptional regulation.',
'MCM' => 'SM00350____minichromosome  maintenance proteins____',
'MD' => 'SM00604________',
'MGS' => 'SM00851____MGS-like domain____This domain composes the whole protein of methylglyoxal synthetase and the domain is also found in Carbamoyl phosphate synthetase (CPS) where it forms a regulatory domain that binds to the allosteric effector ornithine. This family also includes inosicase. The known structures in this family show a common phosphate binding site PUBMED:10526357.',
'MHC_II_alpha' => 'SM00920____Class II histocompatibility antigen, alpha domain____Class II MHC glycoproteins are expressed on the surface of antigen-presenting cells (APC), including macrophages, dendritic cells and B cells. MHC II proteins present peptide antigens that originate extracellularly from foreign bodies such as bacteria. Proteins from the pathogen are degraded into peptide fragments within the APC, which sequesters these fragments into the endosome so they can bind to MHC class II proteins, before being transported to the cell surface. MHC class II receptors display antigens for recognition by helper T cells (stimulate development of B cell clones) and inflammatory T cells (cause the release of lymphokines that attract other cells to site of infection) PUBMED:15120183.',
'MHC_II_beta' => 'SM00921____Class II histocompatibility antigen, beta domain____Class II MHC glycoproteins are expressed on the surface of antigen-presenting cells (APC), including macrophages, dendritic cells and B cells. MHC II proteins present peptide antigens that originate extracellularly from foreign bodies such as bacteria. Proteins from the pathogen are degraded into peptide fragments within the APC, which sequesters these fragments into the endosome so they can bind to MHC class II proteins, before being transported to the cell surface. MHC class II receptors display antigens for recognition by helper T cells (stimulate development of B cell clones) and inflammatory T cells (cause the release of lymphokines that attract other cells to site of infection) PUBMED:15120183.',
'MIF4G' => 'SM00543____Middle domain of eukaryotic initiation factor 4G (eIF4G)____Also occurs in NMD2p and CBP80. The domain is rich in alpha-helices and may contain multiple alpha-helical repeats. In eIF4G, this domain binds eIF4A, eIF3, RNA and DNA. Ponting (TiBS) "Novel eIF4G domain homologues (in press)',
'MIR' => 'SM00472____Domain in ryanodine and inositol trisphosphate receptors and protein O-mannosyltransferases____',
'MIT' => 'SM00745____Microtubule Interacting and Trafficking molecule domain____',
'ML' => 'SM00737____Domain involved in innate immunity and lipid metabolism.____ML (MD-2-related lipid-recognition) is a novel domain identified in MD-1, MD-2, GM2A, Npc2 and multiple proteins of unknown function in plants, animals and fungi. These single-domain proteins were predicted to form a beta-rich fold containing multiple strands, and to mediate diverse biological functions through interacting with specific lipids.',
'MORN' => 'SM00698____Possible plasma membrane-binding motif in junctophilins, PIP-5-kinases and protein kinases.____',
'MR_MLE' => 'SM00922____Mandelate racemase / muconate lactonizing enzyme, C-terminal domain____Mandelate racemase (MR) and muconate lactonizing enzyme (MLE) are two bacterial enzymes involved in aromatic acid catabolism. They catalyze mechanistically distinct reactions yet they are related at the level of their primary, quaternary (homooctamer) and tertiary structures PUBMED:2215699, PUBMED:8256284. This entry represents the C-terminal region of these proteins.',
'MUTSac' => 'SM00534____ATPase domain of DNA mismatch repair MUTS family____',
'MUTSd' => 'SM00533____DNA-binding domain of DNA mismatch repair MUTS family____',
'MYSc' => 'SM00242____Myosin. Large ATPases.____ATPase; molecular motor. Muscle contraction consists of a cyclical interaction between myosin and actin. The core of the myosin structure is similar in fold to that of kinesin.',
'Mad3_BUB1_I' => 'SM00777____Mad3/BUB1 hoMad3/BUB1 homology region 1____Proteins containing this domain are checkpoint proteins involved in cell division. This region has been shown to be essential for the binding of the binding of BUB1 and MAD3 to CDC20p. ',
'Malic_M' => 'SM00919____Malic enzyme, NAD binding domain____Malic enzymes (malate oxidoreductases) catalyse the oxidative decarboxylation of malate to form pyruvate.',
'MbtH' => 'SM00923____MbtH-like protein____This domain is found in the MbtH protein as well as at the N-terminus of the antibiotic synthesis protein NIKP1. This domain is about 70 amino acids long and contains 3 fully conserved tryptophan residues. Many of the members of this family are found in known antibiotic synthesis gene clusters. ',
'MeTrc' => 'SM00138____Methyltransferase, chemotaxis proteins ____Methylates methyl-accepting chemotaxis proteins to form gamma-glutamyl methyl ester residues. ',
'MgtE_N' => 'SM00924____MgtE intracellular N domain____This region is the integral membrane part of the eubacterial MgtE family of magnesium transporters. It is presumed to be an intracellular domain, that may be involved in magnesium binding. ',
'MltA' => 'SM00925____MltA specific insert domain____This beta barrel domain is found inserted in the MltA a murein degrading transglycosylase enzyme. This domain may be involved in peptidoglycan binding.',
'MoCF_biosynth' => 'SM00852____Probable molybdopterin binding domain____This domain is found a variety of proteins involved in biosynthesis of molybdopterin cofactor. The domain is presumed to bind molybdopterin. The structure of this domain is known, and it forms an alpha/beta structure. In the known structure of Gephyrin this domain mediates trimerisation.',
'Molybdop_Fe4S4' => 'SM00926____Molybdopterin oxidoreductase Fe4S4 domain____The molybdopterin oxidoreductase Fe4S4 domain is found in a number of reductase/dehydrogenase families, which include the periplasmic nitrate reductase precursor and the formate dehydrogenase alpha chain.',
'Mterf' => 'SM00733____Mitochondrial termination factor repeats____Human mitochondrial termination factor is a DNA-binding protein that acts as a transcription termination factor. Six repeats occur in human mTERF, that also are present in numerous plant proteins. ',
'MutH' => 'SM00927____DNA mismatch repair enzyme MutH____MutS, MutL and MutH are the three essential proteins for initiation of methyl-directed DNA mismatch repair to correct mistakes made during DNA replication in Escherichia coli. MutH cleaves a newly synthesized and unmethylated daughter strand 5\' to the sequence d(GATC) in a hemi-methylated duplex. Activation of MutH requires the recognition of a DNA mismatch by MutS and MutL PUBMED:9482749. ',
'MutL_C' => 'SM00853____MutL C terminal dimerisation domain____MutL and MutS are key components of the DNA repair machinery that corrects replication errors. MutS recognises mispaired or unpaired bases in a DNA duplex and in the presence of ATP, recruits MutL to form a DNA signaling complex for repair. The N terminal region of MutL contains the ATPase domain and the C terminal is involved in dimerisation. ',
'MyTH4' => 'SM00139____Domain in Myosin and Kinesin Tails____Domain present twice in myosin-VIIa, and also present in 3 other myosins.',
'NADH-G_4Fe-4S_3' => 'SM00929____NADH-ubiquinone oxidoreductase-G iron-sulfur binding region____',
'NADH_4Fe-4S' => 'SM00928____NADH-ubiquinone oxidoreductase-F iron-sulfur binding region____',
'NAT_PEP' => 'SM00183____Natriuretic peptide____Atrial natriuretic peptides are vertebrate hormones important in the overall control of cardiovascular homeostasis and sodium and water balance in general. ',
'NDK' => 'SM00562________These are enzymes that catalyze nonsubstrate specific conversions of nucleoside diphosphates to nucleoside triphosphates. These enzymes play important roles in bacterial growth, signal transduction and pathogenicity.',
'NEAT' => 'SM00725____NEAr Transporter domain____',
'NEBU' => 'SM00227________The Nebulin repeat is present also in Las1. Tandem arrays of these repeats are known to bind actin.',
'NEUZ' => 'SM00588________domain in neuralized proteins',
'NGF' => 'SM00140____Nerve growth factor (NGF or beta-NGF)____NGF is important for the development and maintenance of the  sympathetic and sensory nervous systems. ',
'NGN' => 'SM00738____In Spt5p, this domain may confer affinity for Spt4p. It possesses a RNP-like fold.____In Spt5p, this domain may confer affinity for Spt4p.Spt4p',
'NH' => 'SM00003____Neurohypophysial hormones____Vasopressin/oxytocin gene family. ',
'NIDO' => 'SM00539____Extracellular domain of unknown function in nidogen (entactin) and hypothetical proteins.____',
'NIL' => 'SM00930________This domain is found at the C-terminus of ABC transporter proteins involved in D-methionine transport as well as a number of ferredoxin-like proteins. This domain is likely to act as a substrate binding domain. The domain has been named after a conserved sequence in some members of the family.',
'NL' => 'SM00004____Domain found in Notch and Lin-12____The Notch protein is essential for the proper differentiation of the Drosophila ectoderm. This protein contains 3 NL domains. ',
'NMU' => 'SM00084____Neuromedin U ____ Neuromedin U (NmU) is a vertebrate peptide which stimulates uterine smooth muscle contraction and causes selective vasoconstriction. Like most other active peptides, it is proteolytically processed from a larger precursor protein. The mature peptides are 8 (NmU-8) to 25 (NmU-25) residues long and C- terminally amidated. The sequence of the C-terminal extremity of NmU is extremely well conserved in mammals, birds and amphibians.',
'NOSIC' => 'SM00931____NOSIC (NUC001) domain____This is the central domain in Nop56/SIK1-like proteins PUBMED:15112237.',
'NPCBM' => 'SM00776________This novel putative carbohydrate binding module (NPCBM) domain is found at the N-terminus of glycosyl hydrolase family 98 proteins.',
'NRF' => 'SM00703____N-terminal domain in C. elegans NRF-6 (Nose Resistant to Fluoxetine-4) and NDG-4 (resistant to nordihydroguaiaretic acid-4).____Also present in several other worm and fly proteins.',
'NTR' => 'SM00206____Tissue inhibitor of metalloproteinase family.____Form complexes with metalloproteinases, such as collagenases, and irreversibly inactivate them.',
'NUC' => 'SM00477____DNA/RNA non-specific endonuclease____prokaryotic and eukaryotic double- and single-stranded DNA and RNA endonucleases also present in phosphodiesterases',
'Nfu_N' => 'SM00932____Scaffold protein Nfu/NifU N terminal____This domain is found at the N terminus of NifU and NifU related proteins, and in the human Nfu protein. Both of these proteins are thought to be involved in the the assembly of iron-sulphur clusters PUBMED:12886008.',
'OLF' => 'SM00284____Olfactomedin-like domains____',
'ORANGE' => 'SM00511____Orange domain____This domain confers specificity among members of the Hairy/E(SPL) family.',
'OSTEO' => 'SM00017____Osteopontin____Osteopontin is an acidic phosphorylated glycoprotein of about 40 Kd which is abundant in the mineral matrix of bones and which binds tightly to hydroxyapatite [1,2,3]. It is suggested that osteopontin might function as a cell attachment factor and could play a key role in the adhesion of osteoclasts to the mineral matrix of bone',
'P4Hc' => 'SM00702____Prolyl 4-hydroxylase alpha subunit homologues.____Mammalian enzymes catalyse hydroxylation of collagen, for example. Prokaryotic enzymes might catalyse hydroxylation of antibiotic peptides. These are 2-oxoglutarate-dependent dioxygenases, requiring 2-oxoglutarate and dioxygen as cosubstrates and ferrous iron as a cofactor.',
'PA14' => 'SM00758________domain in bacterial beta-glucosidases other glycosidases, glycosyltransferases, proteases, amidases, yeast adhesins, and bacterial toxins.',
'PA2c' => 'SM00085____Phospholipase A2____',
'PAC' => 'SM00086____Motif C-terminal to PAS motifs (likely to contribute to PAS structural domain)____PAC motif occurs C-terminal to a subset of all known PAS motifs. It is proposed to contribute to the PAS domain fold.',
'PAH' => 'SM00309____Pancreatic hormones / neuropeptide F / peptide YY family ____Pancreatic hormone is a regulator of pancreatic and gastrointestinal functions.',
'PAM' => 'SM00753____PCI/PINT associated module____',
'PAN_AP' => 'SM00473____divergent subfamily of APPLE domains____Apple-like domains present in Plasminogen, C. elegans hypothetical ORFs and the extracellular portion of plant receptor-like protein kinases. Predicted to possess protein- and/or carbohydrate-binding functions.',
'PAS' => 'SM00091____PAS domain____PAS motifs appear in archaea, eubacteria and eukarya. Probably the most surprising identification of a PAS domain was that in EAG-like K+-channels ([1]; Ponting & Aravind, in press).',
'PASTA' => 'SM00740________',
'PAW' => 'SM00613____domain present in PNGases and other hypothetical proteins____present in several copies in proteins with unknown function in C. elegans',
'PAX' => 'SM00351____Paired Box domain____',
'PB1' => 'SM00666____PB1 domain ____Phox and Bem1p domain, present in many eukaryotic cytoplasmic signalling proteins. The domain adopts a beta-grasp fold, similar to that found in ubiquitin and Ras-binding domains. A motif, variously termed OPR, PC and AID, represents the most conserved region of the majority of PB1 domains, and is necessary for PB1 domain function. This function is the formation of PB1 domain heterodimers, although not all PB1 domain pairs associate.',
'PBD' => 'SM00285____P21-Rho-binding domain____Small domains that bind Cdc42p- and/or Rho-like small GTPases. Also known as the Cdc42/Rac interactive binding (CRIB).',
'PBPb' => 'SM00062____Bacterial periplasmic substrate-binding proteins____bacterial proteins, eukaryotic ones are in PBPe',
'PBPe' => 'SM00079____Eukaryotic homologues of bacterial periplasmic substrate binding proteins.____Prokaryotic homologues are represented by a separate alignment: PBPb',
'PD' => 'SM00018____P or trefoil or TFF domain____Proposed role in renewal and pathology of mucous epithelia.',
'PDGF' => 'SM00141____Platelet-derived and vascular endothelial growth factors (PDGF, VEGF) family____Platelet-derived growth factor is a potent activator for cells of  mesenchymal origin. PDGF-A and PDGF-B form AA and BB homodimers and an AB heterodimer. Members of the VEGF family are homologues of PDGF.',
'PDZ' => 'SM00228____Domain present in PSD-95, Dlg, and ZO-1/2.____Also called DHR (Dlg homologous region) or GLGF (relatively well conserved tetrapeptide in these domains). Some PDZs have been shown to bind C-terminal polypeptides; others appear to bind internal (non-C-terminal) polypeptides. Different PDZs possess different binding specificities.',
'PGAM' => 'SM00855____Phosphoglycerate mutase family____Phosphoglycerate mutase (PGAM) and bisphosphoglycerate mutase (BPGM) are structurally related enzymes that catalyse reactions involving the transfer of phospho groups between the three carbon atoms of phosphoglycerate PUBMED:2847721, PUBMED:2831102, PUBMED:10958932. Both enzymes can catalyse three different reactions with different specificities, the isomerization of 2-phosphoglycerate (2-PGA) to 3-phosphoglycerate (3-PGA) with 2,3-diphosphoglycerate (2,3-DPG) as the primer of the reaction, the synthesis of 2,3-DPG from 1,3-DPG with 3-PGA as a primer and the degradation of 2,3-DPG to 3-PGA (phosphatase activity). In mammals, PGAM is a dimeric protein with two isoforms, the M (muscle) and B (brain) forms. In yeast, PGAM is a tetrameric protein. ',
'PGA_cap' => 'SM00854____Bacterial capsule synthesis protein PGA_cap____This protein is a putative poly-gamma-glutamate capsule biosynthesis protein found in bacteria. Poly-gamma-glutamate is a natural polymer that may be involved in virulence and may help bacteria survive in high salt concentrations. It is a surface-associated protein.',
'PGRP' => 'SM00701____Animal peptidoglycan recognition proteins homologous to Bacteriophage T3 lysozyme.____The bacteriophage molecule, but not its moth homologue, has been shown to have N-acetylmuramoyl-L-alanine amidase activity. One member of this family, Tag7, is a cytokine.',
'PH' => 'SM00233____Pleckstrin homology domain.____Domain commonly found in eukaryotic signalling proteins. The domain family possesses multiple functions including the abilities to bind inositol phosphates, and various proteins. PH domains have been found to possess inserted domains (such as in PLC gamma, syntrophins) and to be inserted within other domains. Mutations in Brutons tyrosine kinase (Btk) within its PH domain cause X-linked agammaglobulinaemia (XLA) in patients. Point mutations cluster into the positively charged end of the molecule around the predicted binding site for phosphatidylinositol lipids.',
'PHB' => 'SM00244____prohibitin homologues____prohibitin homologues',
'PHD' => 'SM00249____PHD zinc finger____The plant homeodomain (PHD) finger is a C4HC3 zinc-finger-like motif found in nuclear proteins thought to be involved in epigenetics and chromatin-mediated transcriptional regulation. The PHD finger binds two zinc ions using the so-called \'cross-brace\' motif and is thus structurally related to the <a href="/smart/do_annotation.pl?BLAST=DUMMY&DOMAIN=184">RING finger</a> and the <a href="/smart/do_annotation.pl?BLAST=DUMMY&DOMAIN=64">FYVE finger</a>. It is not yet known if PHD fingers have a common molecular function. Several reports suggest that it can function as a protein-protein interacton domain and it was recently demonstrated that the PHD finger of p300 can cooperate with the adjacent <a href="/smart/do_annotation.pl?BLAST=DUMMY&DOMAIN=297">BROMO domain</a> in nucleosome binding in vitro. Other reports suggesting that the PHD finger is a ubiquitin ligase have been refuted as these domains were <a href="/smart/do_annotation.pl?BLAST=DUMMY&DOMAIN=184">RING fingers</a> misidentified as PHD fingers.',
'PI3K_C2' => 'SM00142____Phosphoinositide 3-kinase, region postulated to contain C2 domain____Outlier of C2 family.',
'PI3K_p85B' => 'SM00143____PI3-kinase family, p85-binding domain____Region of p110 PI3K that binds the p85 subunit.',
'PI3K_rbd' => 'SM00144____PI3-kinase family, Ras-binding domain____Certain members of the PI3K family possess Ras-binding domains in their N-termini. These regions show some similarity (although not highly  significant similarity) to Ras-binding RA domains (unpublished observation).',
'PI3Ka' => 'SM00145____Phosphoinositide 3-kinase family, accessory domain (PIK domain)____PIK domain is conserved in all PI3 and PI4-kinases. Its role is unclear but it has been suggested to be involved in substrate presentation.',
'PI3Kc' => 'SM00146____Phosphoinositide 3-kinase, catalytic domain____Phosphoinositide 3-kinase isoforms participate in a variety of processes,  including cell motility, the Ras pathway, vesicle trafficking and  secretion, and apoptosis. These homologues may be either lipid kinases and/or protein kinases: the former phosphorylate the 3-position in the inositol ring of inositol phospholipids. The ataxia telangiectesia-mutated gene produced, the targets of rapamycin (TOR) and the DNA-dependent kinase have not been found to possess lipid kinase activity. Some of this family possess PI-4 kinase activities.',
'PIG-X' => 'SM00780____PIG-X / PBN1____Mammalian PIG-X and yeast PBN1 are essential components of glycosylphosphatidylinositol-mannosyltransferase I. These enzymes are involved in the transfer of sugar molecules.',
'PINT' => 'SM00088____motif in proteasome subunits, Int-6, Nip-1 and TRIP-15____Also called the PCI (Proteasome, COP9, Initiation factor 3) domain. Unknown function.',
'PINc' => 'SM00670____Large family of predicted nucleotide-binding domains____From similarities to 5\'-exonucleases, these domains are predicted to be RNases. PINc domains in nematode SMG-5 and yeast NMD4p are predicted to be involved in RNAi.',
'PIPKc' => 'SM00330____Phosphatidylinositol phosphate kinases____',
'PKD' => 'SM00089____Repeats in polycystic kidney disease 1 (PKD1) and other proteins____Polycystic kidney disease 1 protein contains 14 repeats, present elsewhere such as in microbial collagenases.',
'PKS_AT' => 'SM00827____Acyl transferase domain in polyketide synthase (PKS) enzymes.____',
'PKS_DH' => 'SM00826________',
'PKS_ER' => 'SM00829____Enoylreductase____Enoylreductase in Polyketide synthases.',
'PKS_KR' => 'SM00822________This enzymatic domain is part of bacterial polyketide synthases and catalyses the first step in the reductive modification of the beta-carbonyl centres in the growing polyketide chain. It uses NADPH to reduce the keto group to a hydroxy group.',
'PKS_KS' => 'SM00825____Beta-ketoacyl synthase____The structure of beta-ketoacyl synthase is similar to that of the thiolase family and also chalcone synthase. The active site of beta-ketoacyl synthase is located between the N and C-terminal domains.',
'PKS_MT' => 'SM00828____Methyltransferase  in polyketide synthase (PKS) enzymes.____',
'PKS_PP' => 'SM00823____Phosphopantetheine attachment site____Phosphopantetheine (or pantetheine 4\' phosphate) is the prosthetic group of acyl carrier proteins (ACP) in some multienzyme complexes where it serves as a \'swinging arm\' for the attachment of activated fatty acid and amino-acid groups PUBMED:5321311.',
'PKS_TE' => 'SM00824____Thioesterase____Peptide synthetases are involved in the non-ribosomal synthesis of peptide antibiotics. Next to the operons encoding these enzymes, in almost all cases, are genes that encode proteins that have similarity to the type II fatty acid thioesterases of vertebrates. There are also modules within the peptide synthetases that also share this similarity. With respect to antibiotic production, thioesterases are required for the addition of the last amino acid to the peptide antibiotic, thereby forming a cyclic antibiotic. Thioesterases (non-integrated) have molecular masses of 25-29 kDa.',
'PLAc' => 'SM00022____Cytoplasmic phospholipase A2, catalytic subunit____Cytosolic phospholipases A2 hydrolyse arachidonyl phospholipids. Family includes phospholipases B isoforms.',
'PLCXc' => 'SM00148____Phospholipase C, catalytic domain (part); domain X____Phosphoinositide-specific phospholipases C. These enzymes contain 2 regions (X and Y) which together form a TIM barrel-like structure containing the active site residues. Phospholipase C enzymes (PI-PLC) act as signal transducers that generate two second messengers,  inositol-1,4,5-trisphosphate and diacylglycerol. The bacterial enzyme [6] appears to be a homologue of the mammalian PLCs.',
'PLCYc' => 'SM00149____Phospholipase C, catalytic domain (part); domain Y____Phosphoinositide-specific phospholipases C. These enzymes contain 2 regions (X and Y) which together form a TIM barrel-like structure containing the active site residues. Phospholipase C enzymes (PI-PLC) act as signal transducers that generate two second messengers,  inositol-1,4,5-trisphosphate and diacylglycerol. The bacterial enzyme [6] appears to be a homologue of the mammalian PLCs.',
'PLDc' => 'SM00155____Phospholipase D. Active site motifs.____Phosphatidylcholine-hydrolyzing phospholipase D (PLD) isoforms are  activated by ADP-ribosylation factors (ARFs). PLD produces phosphatidic  acid from phosphatidylcholine, which may be essential for the formation  of certain types of transport vesicles or may be constitutive vesicular transport to signal transduction pathways.  PC-hydrolysing PLD is a homologue of cardiolipin synthase, phosphatidylserine synthase, bacterial PLDs, and viral proteins. Each of these appears to  possess a domain duplication which is apparent by the presence of two motifs containing well-conserved histidine, lysine, aspartic acid,  and/or asparagine residues which may contribute to the active site. An E. coli endonuclease (nuc) and similar proteins appear to be PLD homologues but possess only one of these motifs. The profile contained here represents only the putative active site regions, since an accurate multiple alignment of the repeat units has not been achieved.',
'PLEC' => 'SM00250____Plectin repeat____',
'PLP' => 'SM00002____Myelin proteolipid protein (PLP or lipophilin)____',
'PMEI' => 'SM00856____Plant invertase/pectin methylesterase inhibitor____This domain inhibits pectin methylesterases (PMEs) and invertases through formation of a non-covalent 1:1 complex PUBMED:8521860. It has been implicated in the regulation of fruit development, carbohydrate metabolism and cell wall extension. It may also be involved in inhibiting microbial pathogen PMEs. It has been observed that it is often expressed as a large inactive preprotein PUBMED:8521860. It is also found at the N-termini of PMEs predicted from DNA sequences, suggesting that both PMEs and their inhibitors are expressed as a single polyprotein and subsequently processed. It has two disulphide bridges and is mainly alpha-helical PUBMED:10880981.',
'POL3Bc' => 'SM00480____DNA polymerase III beta subunit____',
'POLAc' => 'SM00482____DNA polymerase A domain____',
'POLBc' => 'SM00486____DNA polymerase type-B family____DNA polymerase alpha, delta, epsilon and zeta chain (eukaryota), DNA polymerases in archaea, DNA polymerase II in e. coli, mitochondrial DNA polymerases and and virus DNA polymerases',
'POLIIIAc' => 'SM00481____DNA polymerase alpha chain like domain____DNA polymerase alpha chain like domain, incl. family of hypothetical proteins',
'POLXc' => 'SM00483____DNA polymerase X family____includes vertebrate polymerase beta and terminal deoxynucleotidyltransferases',
'POP4' => 'SM00538____A domain found in a protein subunit of human RNase MRP and RNase P ribonucleoprotein complexes and archaeal proteins.____',
'POU' => 'SM00352____Found in Pit-Oct-Unc transcription factors____',
'POX' => 'SM00574____domain associated with HOX domains____',
'PP2Ac' => 'SM00156____Protein phosphatase 2A homologues, catalytic domain.____Large family of serine/threonine phosphatases, that includes PP1, PP2A and PP2B (calcineurin) family members.',
'PP2C_SIG' => 'SM00331____Sigma factor PP2C-like phosphatases____',
'PP2Cc' => 'SM00332____Serine/threonine phosphatases, family 2C, catalytic domain____The protein architecture and deduced catalytic mechanism of PP2C phosphatases are similar to the PP1, PP2A, PP2B family of protein Ser/Thr phosphatases, with which PP2C shares no sequence similarity.',
'PQQ' => 'SM00564____beta-propeller repeat____Beta-propeller repeat occurring in enzymes with pyrrolo-quinoline quinone (PQQ) as cofactor, in Ire1p-like Ser/Thr kinases, and in prokaryotic dehydrogenases.',
'PRE_C2HC' => 'SM00596________',
'PROF' => 'SM00392____Profilin____Binds actin monomers, membrane polyphosphoinositides and poly-L-proline. ',
'PRP' => 'SM00157____Major prion protein____The prion protein is a major component of scrapie-associated fibrils in  Creutzfeldt-Jakob disease, kuru, Gerstmann-Straussler syndrome and bovine spongiform encephalopathy.',
'PRY' => 'SM00589________associated with SPRY domains',
'PSA' => 'SM00639____Paramecium Surface Antigen Repeat____',
'PSI' => 'SM00423____domain found in Plexins, Semaphorins and Integrins____',
'PSN' => 'SM00730____Presenilin, signal peptide peptidase, family____Presenilin 1 and presenilin 2 are polytopic membrane proteins, whose genes are mutated in some individuals with Alzheimer\'s disease. Distant homologues, present in eukaryotes and archaea, also contain conserved aspartic acid residues which are predicted to contribute to catalysis. At least one member of this family has been shown to possess signal peptide peptidase activity. ',
'PSP' => 'SM00581____proline-rich domain in spliceosome associated proteins____',
'PTB' => 'SM00462____Phosphotyrosine-binding domain, phosphotyrosine-interaction (PI) domain____PTB/PI domain structure similar to those of pleckstrin homology (PH) and IRS-1-like PTB domains.',
'PTBI' => 'SM00310____Phosphotyrosine-binding domain (IRS1-like)____',
'PTH' => 'SM00087____Parathyroid hormone____',
'PTI' => 'SM00286____Plant trypsin inhibitors____',
'PTN' => 'SM00193____Pleiotrophin / midkine family____Heparin-binding domain family.',
'PTPc' => 'SM00194____Protein tyrosine phosphatase, catalytic domain____',
'PTPc_DSPc' => 'SM00012____Protein tyrosine phosphatase, catalytic domain, undefined specificity____Protein tyrosine phosphatases. Homologues detected by this profile and not by those of "PTPc" or  "DSPc" are predicted to be protein phosphatases with a similar fold to DSPs and PTPs, yet with unpredicted specificities.',
'PTPc_motif' => 'SM00404____Protein tyrosine phosphatase, catalytic domain motif____',
'PTX' => 'SM00159____Pentraxin / C-reactive protein / pentaxin family____This family form a doscoid pentameric structure. Human serum amyloid P demonstrates calcium-mediated ligand-binding.',
'PUA' => 'SM00359____Putative RNA-binding Domain in PseudoUridine synthase and Archaeosine transglycosylase____',
'PUG' => 'SM00580____domain in protein kinases, N-glycanases and other nuclear proteins____',
'PUR' => 'SM00712____DNA/RNA-binding repeats in PUR-alpha/beta/gamma and in hypothetical proteins from spirochetes and the Bacteroides-Cytophaga-Flexibacter bacteria.____',
'PWI' => 'SM00311____PWI, domain in splicing factors____',
'PWWP' => 'SM00293____domain with conserved PWWP motif____conservation of Pro-Trp-Trp-Pro residues',
'PX' => 'SM00312____PhoX homologous domain, present in p47phox and p40phox.____Eukaryotic domain of unknown function present in phox proteins, PLD isoforms, a PI3K isoform. ',
'PXA' => 'SM00313____Domain associated with PX domains____unpubl. observations',
'ParB' => 'SM00470____ParB-like nuclease domain____Plasmid RK2 ParB preferentially cleaves single-stranded DNA. ParB also nicks supercoiled plasmid DNA preferably at sites with potential single-stranded character, like AT-rich regions and sequences that can form cruciform structures. ParB also exhibits 5-->3 exonuclease activity.',
'PbH1' => 'SM00710____Parallel beta-helix repeats____The tertiary structures of pectate lyases and rhamnogalacturonase A show a stack of parallel beta strands that are coiled into a large helix. Each coil of the helix represents a structural repeat that, in some homologues, can be recognised from sequence information alone. Conservation of asparagines might be connected with asparagine-ladders that contribute to the stability of the fold. Proteins containing these repeats most often are enzymes with polysaccharide substrates.',
'Pept_C1' => 'SM00645____Papain family cysteine protease____',
'PhBP' => 'SM00708____Insect pheromone/odorant binding protein domains.____',
'PhnA_Zn_Ribbon' => 'SM00782____PhnA Zinc-Ribbon____This protein family includes an uncharacterised member designated phnA in Escherichia coli, part of a large operon associated with alkylphosphonate uptake and carbon-phosphorus bond cleavage. This protein is not related to the characterised phosphonoacetate hydrolase designated PhnA.',
'PlsC' => 'SM00563____Phosphate acyltransferases____Function in phospholipid biosynthesis and have either glycerolphosphate, 1-acylglycerolphosphate, or 2-acylglycerolphosphoethanolamine acyltransferase activities. Tafazzin, the product of the gene mutated in patients with Barth syndrome, is a member of this family.',
'Plus3' => 'SM00719____Short conserved domain in transcriptional regulators.____Plus3 domains occur in the Saccharomyces cerevisiae Rtf1p protein, which interacts with Spt6p, and in parsley CIP, which interacts with the bZIP protein CPRF1.',
'PolyA' => 'SM00517____C-terminal domain of Poly(A)-binding protein. Present also in Drosophila hyperplastics discs protein.____Involved in homodimerisation (either directly or indirectly)',
'PostSET' => 'SM00508____Cysteine-rich motif following a subset of SET domains____',
'PreSET' => 'SM00468____N-terminal to some SET domains____A Cys-rich putative Zn2+-binding domain that occurs N-terminal to some SET domains. Function is unknown. Unpublished.',
'Prim_Zn_Ribbon' => 'SM00778____Zinc-binding domain of primase-helicase____This region represents the zinc binding domain. It is found in the N-terminal region of the bacteriophage P4 alpha protein, which is a multifunctional protein with origin recognition, helicase and primase activities.',
'Pumilio' => 'SM00025____Pumilio-like repeats____Pumilio-like repeats that bind RNA.',
'R3H' => 'SM00393____Putative single-stranded nucleic acids-binding domain____',
'RA' => 'SM00314____Ras association (RalGDS/AF-6) domain____RasGTP effectors (in cases of AF6, canoe and RalGDS); putative RasGTP effectors in other cases. Kalhammer et al. have shown that not all RA domains bind RasGTP. Predicted structure similar to that determined, and that of the RasGTP-binding domain of Raf kinase. Predicted RA domains in PLC210 and nore1 found to bind RasGTP. Included outliers (Grb7, Grb14, adenylyl cyclases etc.)',
'RAB' => 'SM00175____Rab subfamily of small GTPases____Rab GTPases are implicated in vesicle trafficking.',
'RAN' => 'SM00176____Ran (Ras-related nuclear proteins) /TC4 subfamily of small GTPases____Ran is involved in the active transport of proteins through nuclear pores.',
'RAS' => 'SM00173____Ras subfamily of RAS small GTPases____Similar in fold and function to the bacterial EF-Tu GTPase. p21Ras couples receptor Tyr kinases and G protein receptors  to protein kinase cascades',
'RBD' => 'SM00455____Raf-like Ras-binding domain____',
'REC' => 'SM00448____cheY-homologous receiver domain____CheY regulates the clockwise rotation of E. coli flagellar motors. This domain contains a phosphoacceptor site that is phosphorylated by histidine kinase homologues.',
'RGS' => 'SM00315____Regulator of G protein signalling domain____RGS family members are GTPase-activating proteins for heterotrimeric G-protein alpha-subunits.',
'RHO' => 'SM00174____Rho (Ras homology) subfamily of Ras-like small GTPases____Members of this subfamily of Ras-like small GTPases include Cdc42 and Rac, as well as Rho isoforms.',
'RHOD' => 'SM00450____Rhodanese Homology Domain____An alpha beta fold found duplicated in the Rhodanese protein. The the Cysteine containing enzymatically active version of the domain is also found in the CDC25 class of protein phosphatases and a variety of proteins such as sulfide dehydrogenases and stress proteins such as Senesence specific protein 1 in plants, PspE and GlpE in bacteria and cyanide and arsenate resistance proteins. Inactive versions with a loss of the cysteine are also seen in Dual specificity phosphatases, ubiquitin hydrolases from yeast and in sulfuryltransferases. These are likely to play a role in protein interactions. ',
'RIBOc' => 'SM00535____Ribonuclease III family____',
'RICIN' => 'SM00458____Ricin-type beta-trefoil____Carbohydrate-binding domain formed from presumed gene triplication.',
'RIIa' => 'SM00394____RIIalpha, Regulatory subunit portion of type II PKA R-subunit____RIIalpha, Regulatory subunit portion of type II PKA R-subunit. Contains dimerisation interface and binding site for A-kinase-anchoring proteins (AKAPs).',
'RING' => 'SM00184____Ring finger____E3 ubiquitin-protein ligase activity is intrinsic to the RING domain of c-Cbl and is likely to be a general function of this domain; Various RING fingers exhibit binding activity towards E2 ubiquitin-conjugating enzymes (Ubc\' s)',
'RINGv' => 'SM00744____The RING-variant domain is a C4HC3 zinc-finger like motif found in a number of cellular and viral proteins. Some of these proteins have been shown both in vivo and in vitro to have ubiquitin E3 ligase activity.____The RING-variant domain is reminiscent of both the RING and the PHD domains and may represent an evolutionary intermediate. To describe this domain the term PHD/LAP domain has been used in the past. Extended description: The RING-variant (RINGv) domain contains a C4HC3 zinc-finger-like motif similar to the PHD domain, while some of the spacing between the Cys/His residues follow a pattern somewhat closer to that found in the RING domain. The RINGv domain, similar to the RING, PHD and LIM domains, is thought to bind two zinc ions co-ordinated by the highly conserved Cys and His residues. RING variant domain: C-x (2) -C-x(10-45)-C-x (1) -C-x (7) -H-x(2)-C-x(11-25)-C-x(2)-C As opposed to a PHD: C-x(1-2) -C-x (7-13)-C-x(2-4)-C-x(4-5)-H-x(2)-C-x(10-21)-C-x(2)-C Classical RING domain: C-x (2) -C-x (9-39)-C-x(1-3)-H-x(2-3)-C-x(2)-C-x(4-48) -C-x(2)-C',
'RIO' => 'SM00090____RIO-like kinase____',
'RL11' => 'SM00649____Ribosomal protein L11/L12____',
'RNAse_Pc' => 'SM00092____Pancreatic ribonuclease ____',
'RPEL' => 'SM00707____Repeat in Drosophila CG10860, human KIAA0680 and C. elegans F26H9.2____',
'RPOL4c' => 'SM00657____DNA-directed RNA-polymerase II subunit ____',
'RPOL8c' => 'SM00658____RNA polymerase subunit 8____subunit of RNA polymerase I, II and III',
'RPOL9' => 'SM00661____RNA polymerase subunit 9____',
'RPOLA_N' => 'SM00663____RNA polymerase I subunit A N-terminus____',
'RPOLCX' => 'SM00659____RNA polymerase subunit CX____present in RNA polymerase I, II and III',
'RPOLD' => 'SM00662____RNA polymerases D____DNA-directed RNA polymerase subunit D and bacterial alpha chain',
'RPR' => 'SM00582________domain present in proteins, which are involved in regulation of nuclear pre-mRNA',
'RRM' => 'SM00360____RNA recognition motif____',
'RRM_1' => 'SM00361____RNA recognition motif____',
'RRM_2' => 'SM00362____RNA recognition motif____',
'RUN' => 'SM00593________domain involved in Ras-like GTPase signaling',
'RWD' => 'SM00591________domain in RING finger and WD repeat containing proteins and DEXDc-like helicases subfamily related to the UBCc domain',
'RanBD' => 'SM00160____Ran-binding domain____Domain of apporximately 150 residues that stabilises the GTP-bound form of Ran (the Ras-like nuclear small GTPase).',
'RasGAP' => 'SM00323____GTPase-activator protein for Ras-like GTPases____All alpha-helical domain that accelerates the GTPase activity of Ras, thereby "switching" it into an "off" position. Improved domain limits from structure.',
'RasGEF' => 'SM00147____Guanine nucleotide exchange factor for Ras-like small GTPases____',
'RasGEFN' => 'SM00229____Guanine nucleotide exchange factor for Ras-like GTPases; N-terminal motif____A subset of guanine nucleotide exchange factor for Ras-like small GTPases appear to possess this domain N-terminal to the RasGef (Cdc25-like) domain. The recent crystal structureof Sos shows that this domain is alpha-helical and plays a "purely structural role" (Nature 394, 337-343).',
'Resolvase' => 'SM00857____ Resolvase, N terminal domain____The N-terminal domain of the resolvase family contains the active site and the dimer interface. The extended arm at the C-terminus of this domain connects to the C-terminal helix-turn-helix domain of resolvase.',
'RhoGAP' => 'SM00324____GTPase-activator protein for Rho-like GTPases____GTPase activator proteins towards Rho/Rac/Cdc42-like small GTPases. etter domain limits and outliers.',
'RhoGEF' => 'SM00325____Guanine nucleotide exchange factor for Rho/Rac/Cdc42-like GTPases____Guanine nucleotide exchange factor for Rho/Rac/Cdc42-like GTPases Also called Dbl-homologous (DH) domain. It appears that PH domains invariably occur C-terminal to RhoGEF/DH domains. Improved coverage.',
'S1' => 'SM00316____Ribosomal protein S1-like RNA-binding domain____',
'S4' => 'SM00363____S4 RNA-binding domain____',
'SAA' => 'SM00197____Serum amyloid A proteins____Serum amyloid A proteins are induced during the acute-phase response. Secondary amyloidosis is characterised by the extracellular accumulation in tissues of SAA proteins. SAA proteins are apolipoproteins.',
'SAF' => 'SM00858________This domain family includes a range of different proteins. Such as antifreeze proteins and flagellar FlgA proteins, and CpaB pilus proteins.',
'SAM' => 'SM00454____Sterile alpha motif.____Widespread domain in signalling and nuclear proteins. In EPH-related tyrosine kinases, appears to mediate cell-cell initiated signal transduction via the binding of SH2-containing proteins to a conserved tyrosine that is phosphorylated. In many cases mediates homodimerisation.',
'SAM_PNT' => 'SM00251____SAM / Pointed domain____A subfamily of the SAM domain',
'SAND' => 'SM00258____SAND domain____',
'SANT' => 'SM00717____SANT  SWI3, ADA2, N-CoR and TFIIIB\'\' DNA-binding domains____',
'SAP' => 'SM00513____Putative DNA-binding (bihelical) motif predicted to be involved in chromosomal organisation____',
'SAPA' => 'SM00162____Saposin/surfactant protein-B A-type DOMAIN____Present as four and three degenerate copies, respectively, in prosaposin and surfactant protein B. Single copies in acid sphingomyelinase, NK-lysin amoebapores and granulysin. Putative phospholipid membrane binding domains. ',
'SAR' => 'SM00178____Sar1p-like members of the Ras-family  of small GTPases____Yeast SAR1 is an essential gene required for transport of secretory proteins from the endoplasmic reticulum to the Golgi apparatus.',
'SCAN' => 'SM00431____leucine rich region____',
'SCP' => 'SM00198____SCP / Tpx-1 / Ag5 / PR-1 / Sc7 family of extracellular domains.____Human glioma pathogenesis-related protein GliPR and the plant  pathogenesis-related protein represent functional links between plant defense systems and human immune system. This family has no known function.',
'SCY' => 'SM00199____Intercrine alpha family (small cytokine C-X-C) (chemokine CXC).____Family of cytokines involved in cell-specific chemotaxis,  mediation of cell growth, and the inflammatory response.',
'SEA' => 'SM00200____Domain found in sea urchin sperm protein, enterokinase, agrin____Proposed function of regulating or binding carbohydrate sidechains.',
'SEC14' => 'SM00516____Domain in homologues of a S. cerevisiae phosphatidylinositol transfer protein (Sec14p)____Domain in homologues of a S. cerevisiae phosphatidylinositol transfer protein (Sec14p) and in RhoGAPs, RhoGEFs and the RasGAP, neurofibromin (NF1). Lipid-binding domain. The SEC14 domain of Dbl is known to associate with G protein beta/gamma subunits.',
'SEC63' => 'SM00611____Domain of unknown function in Sec63p, Brr2p and other proteins.____',
'SEL1' => 'SM00671____Sel1-like repeats.____These represent a subfamily of TPR (tetratricopeptide repeat) sequences.',
'SEP' => 'SM00553____Domain present in Saccharomyces cerevisiae Shp1, Drosophila melanogaster eyes closed gene (eyc), and vertebrate p47.____',
'SERPIN' => 'SM00093____SERine  Proteinase INhibitors____',
'SET' => 'SM00317____SET (Su(var)3-9, Enhancer-of-zeste, Trithorax) domain____Putative methyl transferase, based on outlier plant homologues',
'SFM' => 'SM00500____Splicing Factor Motif, present in Prp18 and Pr04____',
'SF_P' => 'SM00019____Pulmonary surfactant proteins____Pulmonary surfactant associated proteins promote alveolar stability by lowering the surface tension at the air-liquid interface in the peripheral air spaces. SP-C, a component of surfactant, is a highly hydrophobic peptide of 35 amino acid residues which is processed from a larger precursor protein. SP-C is post-translationally modified by the covalent attachment of two palmitoyl groups on two adjacent cysteines',
'SH2' => 'SM00252____Src homology 2 domains____Src homology 2 domains bind phosphotyrosine-containing polypeptides via 2 surface pockets. Specificity is provided via interaction with residues that are distinct from the phosphotyrosine. Only a single occurrence of a SH2 domain has been found in S. cerevisiae.',
'SH3' => 'SM00326____Src homology 3 domains____Src homology 3 (SH3) domains bind to target proteins through sequences containing proline and hydrophobic amino acids. Pro-containing polypeptides may bind to SH3 domains in 2 different binding orientations.',
'SH3b' => 'SM00287____Bacterial SH3 domain homologues____',
'SHR3_chaperone' => 'SM00786____ER membrane protein SH3____This family of proteins are membrane localised chaperones that are required for correct plasma membrane localisation of amino acid permeases (AAPs) PUBMED:15623581. Shr3 prevents AAPs proteins from aggregating and assists in their correct folding. In the absence of Shr3, AAPs are retained in the ER.',
'SMI1_KNR4' => 'SM00860____SMI1 / KNR4 family____Proteins in this family are involved in the regulation of 1,3-beta-glucan synthase activity and cell-wall formation.',
'SMR' => 'SM00463____Small MutS-related domain____',
'SNc' => 'SM00318____Staphylococcal nuclease homologues____',
'SO' => 'SM00201____Somatomedin B -like domains____Somatomedin-B is a peptide, proteolytically excised from vitronectin, that is a growth hormone-dependent serum factor with protease-inhibiting activity.',
'SOCS' => 'SM00253____suppressors of cytokine signalling____suppressors of cytokine signalling',
'SPEC' => 'SM00150____Spectrin repeats____',
'SPK' => 'SM00583________domain in SET and PHD domain containing proteins and protein kinases',
'SPRY' => 'SM00449____Domain in SPla and the RYanodine Receptor. ____Domain of unknown function. Distant homologues are domains in butyrophilin/marenostrin/pyrin homologues.',
'SPT2' => 'SM00784____SPT2 chromatin protein____This entry includes the Saccharomyces cerevisiae protein SPT2 which is a chromatin protein involved in transcriptional regulation PUBMED:15563464.',
'SR' => 'SM00202____Scavenger receptor Cys-rich____The sea ucrhin egg peptide speract contains 4 repeats of SR domains that contain 6 conserved cysteines. May bind bacterial antigens in the protein MARCO.',
'SRA' => 'SM00466____SET and RING finger associated domain. Domain of unknown function in SET domain containing proteins and in Deinococcus radiodurans DRA1533.____Domain in SET domain containing proteins and in Deinococcus radiodurans DRA1533. ',
'START' => 'SM00234____in StAR and phosphatidylcholine transfer protein____putative lipid-binding domain in StAR and phosphatidylcholine transfer protein',
'STE' => 'SM00424____STE like transcription factors____',
'STI' => 'SM00452____Soybean trypsin inhibitor (Kunitz) family of protease inhibitors____',
'STI1' => 'SM00727____Heat shock chaperonin-binding motif.____',
'STYKc' => 'SM00221____Protein kinase; unclassified specificity.____Phosphotransferases. The specificity of this class of kinases can not be predicted. Possible dual-specificity Ser/Thr/Tyr kinase.',
'SWAP' => 'SM00648____Suppressor-of-White-APricot splicing regulator____domain present in regulators which are responsible for pre-mRNA splicing processes ',
'SWIB' => 'SM00151____SWI complex, BAF60b domains____',
'S_TK_X' => 'SM00133____Extension to Ser/Thr-type protein kinases____',
'S_TKc' => 'SM00220____Serine/Threonine protein kinases, catalytic domain____Phosphotransferases. Serine or threonine-specific kinase subfamily.',
'SapB' => 'SM00741____Saposin (B) Domains____Present in multiple copies in prosaposin and in pulmonary surfactant-associated protein B. In plant aspartic proteinases, a saposin domain is circularly permuted. This causes the prediction algorithm to predict two such domains, where only one is truly present.',
'Sec7' => 'SM00222____Sec7 domain____Domain named after the S. cerevisiae SEC7 gene product, which is required for proper protein transport through the Golgi. The domain facilitates guanine nucleotide exchange on the small GTPases, ARFs (ADP ribosylation factors).',
'Sema' => 'SM00630____semaphorin domain____',
'Semialdhyde_dh' => 'SM00859____Semialdehyde dehydrogenase, NAD binding domain____The semialdehyde dehydrogenase family is found in N-acetyl-glutamine semialdehyde dehydrogenase (AgrC), which is involved in arginine biosynthesis, and aspartate-semialdehyde dehydrogenase, an enzyme involved in the biosynthesis of various amino acids from aspartate. This family is also found in yeast and fungal Arg5,6 protein, which is cleaved into the enzymes N-acety-gamma-glutamyl-phosphate reductase and acetylglutamate kinase. These are also involved in arginine biosynthesis. All proteins in this entry contain a NAD binding region of semialdehyde dehydrogenase.',
'ShKT' => 'SM00254____ShK toxin domain____ShK toxin domain',
'Skp1' => 'SM00512____Found in Skp1 protein family____Family of Skp1 (kinetochore protein required for cell cycle progression) and elongin C (subunit of RNA polymerase II transcription factor SIII) homologues.',
'Sm' => 'SM00651____snRNP Sm proteins____small nuclear ribonucleoprotein particles (snRNPs) involved in pre-mRNA splicing',
'Sorb' => 'SM00459____Sorbin homologous domain____First found in the peptide hormone sorbin and later in the ponsin/ArgBP2/vinexin family of proteins.',
'Spc7' => 'SM00787____Spc7 kinetochore protein____This domain is found in cell division proteins which are required for kinetochore-spindle association.',
'SprT' => 'SM00731____SprT homologues.____Predicted to have roles in transcription elongation. Contains a conserved HExxH motif, indicating a metalloprotease function.',
'SynN' => 'SM00503____Syntaxin N-terminal domain____Three-helix domain that (in Sso1p) slows the rate of its reaction with the SNAP-25 homologue Sec9p',
'TAF' => 'SM00803____TATA box binding protein associated factor____TAFs (TATA box binding protein associated factors) are part of the transcription initiation factor TFIID multimeric protein complex. TFIID is composed of the TATA box binding protein (TBP) and a number of TAFs. The TAFs provide binding sites for many different transcriptional activators and co-activators that modulate transcription initiation by Pol II. TAF proteins adopt a histone-like fold.',
'TAFH' => 'SM00549____TAF homology____Domain in Drosophila nervy, CBFA2T1, human TAF105, human TAF130, and Drosophila TAF110. Also known as nervy homology region 1 (NHR1).',
'TAP_C' => 'SM00804____C-terminal domain of vertebrate Tap protein____The vertebrate Tap protein is a member of the NXF family of shuttling transport receptors for the nuclear export of mRNA. Its most C-terminal domain is important for binding to FG repeat-containing nuclear pore proteins (FG-nucleoporins) and is sufficient to mediate shuttling. This domain forms a compact four-helix fold related to that of a UBA domain. ',
'TBC' => 'SM00164____Domain in Tre-2, BUB2p, and Cdc16p. Probable Rab-GAPs.____Widespread domain present in Gyp6 and Gyp7, thereby giving rise to the notion that it performs a GTP-activator activity on Rab-like GTPases.',
'TBOX' => 'SM00425____Domain first found  in the mice T locus (Brachyury) protein____',
'TDU' => 'SM00711____Short repeats in human TONDU, fly vestigial and other proteins.____Unknown function. ',
'TEA' => 'SM00426____TEA domain____',
'TECPR' => 'SM00706____Beta propeller repeats in Physarum polycephalum tectonins, Limulus lectin L-6 and animal hypothetical proteins.____',
'TFIIE' => 'SM00531____Transcription initiation factor IIE____',
'TFS2M' => 'SM00510____Domain in the central regions of transcription elongation factor S-II (and elsewhere)____',
'TFS2N' => 'SM00509____Domain in the N-terminus of transcription elongation factor S-II (and elsewhere) ____',
'TGFB' => 'SM00204____Transforming growth  factor-beta (TGF-beta) family____Family members are active as disulphide-linked homo- or heterodimers. TGFB is a multifunctional peptide that controls proliferation,  differentiation, and other functions in many cell types.',
'TGc' => 'SM00460____Transglutaminase/protease-like homologues____Transglutaminases are enzymes that establish covalent links between proteins. A subset of transglutaminase homologues appear to catalyse the reverse reaction, the hydrolysis of peptide bonds. Proteins with this domain are both extracellular and intracellular, and it is likely that the eukaryotic intracellular proteins are involved in signalling events.',
'THEG' => 'SM00705____Repeats in THEG (testicular haploid expressed gene) and several fly proteins.____',
'THN' => 'SM00205____Thaumatin family____The thaumatin family gathers proteins related to plant pathogenesis. The thaumatin family includes very basic members with extracellular and vacuolar localization. Thaumatin itsel is a potent sweet-tasting protein. Several members of this family display significant in vitro activity of inhibiting hyphal growth or spore germination of various fungi probably by a membrane permeabilizing mechanism. ',
'THY' => 'SM00152____Thymosin beta actin-binding motif.____',
'TIR' => 'SM00255____Toll - interleukin 1 - resistance____',
'TK' => 'SM00203____Tachykinin family____Tachykinins are a group of biologically active peptides which excite neurons, evoke behavioral responses, are potent vasodilatators and contract (directly or indirectly) many smooth muscles. These peptides are synthesized as longer precursors and then processed to peptides from ten to twelve residues long. ',
'TLC' => 'SM00724____TRAM, LAG1 and CLN8 homology domains.____Protein domain with at least 5 transmembrane alpha-helices. Lag1p and Lac1p are essential for acyl-CoA-dependent ceramide synthesis, TRAM is a subunit of the translocon and the CLN8 gene is mutated in Northern epilepsy syndrome. The family may possess multiple functions such as lipid trafficking, metabolism, or sensing. Trh homologues possess additional homeobox domains.',
'TLDc' => 'SM00584____domain in TBC and LysM domain containing proteins____',
'TNF' => 'SM00207____Tumour necrosis factor family.____Family of cytokines that form homotrimeric or heterotrimeric complexes. TNF mediates mature T-cell receptor-induced apoptosis through the p75  TNF receptor.',
'TNFR' => 'SM00208____Tumor necrosis factor receptor / nerve growth factor receptor repeats.____Repeats in growth factor receptors that are involved in growth factor binding. TNF/TNFR',
'TOP1Ac' => 'SM00437____Bacterial DNA topoisomerase I DNA-binding domain____Bacterial DNA topoisomerase I and III, Eukaryotic DNA topoisomeraes III, reverse gyrase alpha subunit',
'TOP1Bc' => 'SM00436____Bacterial DNA topoisomeraes I ATP-binding domain____Extension of TOPRIM in Bacterial DNA topoisomeraes I and III, Eukaryotic DNA topoisomeraes III, reverse gyrase beta subunit',
'TOP2c' => 'SM00433____TopoisomeraseII____Eukaryotic DNA topoisomerase II, GyrB, ParE',
'TOP4c' => 'SM00434____DNA Topoisomerase IV____Bacterial DNA topoisomerase IV, GyrA, ParC',
'TOPEUc' => 'SM00435____DNA Topoisomerase I (eukaryota)____DNA Topoisomerase I (eukaryota), DNA topoisomerase V, Vaccina virus topoisomerase, Variola virus topoisomerase, Shope fibroma virus topoisomeras',
'TOPRIM' => 'SM00493________topoisomerases, DnaG-type primases, OLD family nucleases and RecR proteins',
'TPR' => 'SM00028____Tetratricopeptide repeats____Repeats present in 4 or more copies in proteins. Contain a minimum of 34 amino acids each and self-associate via a "knobs and holes" mechanism.',
'TRASH' => 'SM00746____metallochaperone-like domain____',
'TR_FER' => 'SM00094____Transferrin____',
'TR_THY' => 'SM00095____Transthyretin____',
'TSP1' => 'SM00209____Thrombospondin type 1 repeats____Type 1 repeats in thrombospondin-1 bind and activate TGF-beta.',
'TSPN' => 'SM00210____Thrombospondin N-terminal -like domains.____Heparin-binding and cell adhesion domain of thrombospondin',
'TSPc' => 'SM00245____tail specific protease____tail specific protease',
'TUDOR' => 'SM00333____Tudor domain____Domain of unknown function present in several RNA-binding proteins. 10 copies in the Drosophila Tudor protein. Initial proposal that the survival motor neuron gene product contain a Tudor domain are corroborated by more recent database search techniques such as PSI-BLAST (unpublished).',
'TY' => 'SM00211____Thyroglobulin type I repeats.____The N-terminal region of human thyroglobulin contains 11 type-1 repeats TY repeats are proposed to be inhibitors of cysteine proteases and binding partners of heparin.',
'TarH' => 'SM00319____Homologues of the ligand binding domain of Tar____Homologues of the ligand binding domain of the wild-type bacterial aspartate receptor, Tar.',
'Trans_reg_C' => 'SM00862____Transcriptional regulatory protein, C terminal____This domain is almost always found associated with the response regulator receiver domain. It may play a role in DNA binding.',
'Transket_pyr' => 'SM00861____Transketolase, pyrimidine binding domain____Transketolase (TK) catalyzes the reversible transfer of a two-carbon ketol unit from xylulose 5-phosphate to an aldose receptor, such as ribose 5-phosphate, to form sedoheptulose 7-phosphate and glyceraldehyde 3- phosphate. This enzyme, together with transaldolase, provides a link between the glycolytic and pentose-phosphate pathways. TK requires thiamine pyrophosphate as a cofactor. In most sources where TK has been purified, it is a homodimer of approximately 70 Kd subunits. TK sequences from a variety of eukaryotic and prokaryotic sources show that the enzyme has been evolutionarily conserved. In the peroxisomes of methylotrophic yeast Hansenula polymorpha, there is a highly related enzyme, dihydroxy-acetone synthase (DHAS) (also known as formaldehyde transketolase), which exhibits a very unusual specificity by including formaldehyde amongst its substrates.',
'Tryp_SPc' => 'SM00020____Trypsin-like serine protease____Many of these are synthesised as inactive precursor zymogens that are cleaved during limited proteolysis to generate their active forms. A few, however, are active as single chain molecules, and others are inactive due to substitutions of the catalytic triad residues.',
'Tubulin' => 'SM00864____Tubulin/FtsZ family, GTPase domain____This domain is found in all tubulin chains, as well as the bacterial FtsZ family of proteins. These proteins are involved in polymer formation. Tubulin is the major component of microtubules, while FtsZ is the polymer-forming protein of bacterial cell division, it is part of a ring in the middle of the dividing cell that is required for constriction of cell membrane and cell envelope to yield two daughter cells. FtsZ and tubulin are GTPases, this entry is the GTPase domain. FtsZ can polymerise into tubes, sheets, and rings in vitro and is ubiquitous in bacteria and archaea.',
'Tubulin_C' => 'SM00865____Tubulin/FtsZ family, C-terminal domain____This domain is found in the tubulin alpha, beta and gamma chains, as well as the bacterial FtsZ family of proteins. These proteins are GTPases and are involved in polymer formation. Tubulin is the major component of microtubules, while FtsZ is the polymer-forming protein of bacterial cell division, it is part of a ring in the middle of the dividing cell that is required for constriction of cell membrane and cell envelope to yield two daughter cells. FtsZ can polymerise into tubes, sheets, and rings in vitro and is ubiquitous in bacteria and archaea. This is the C-terminal domain.',
'TyrKc' => 'SM00219____Tyrosine kinase, catalytic domain____Phosphotransferases. Tyrosine-specific kinase subfamily.',
'UAS' => 'SM00594________',
'UBA' => 'SM00165____Ubiquitin associated domain____Present in Rad23, SNF1-like kinases. The newly-found UBA in p62 is known to bind ubiquitin.',
'UBCc' => 'SM00212____Ubiquitin-conjugating enzyme E2, catalytic domain homologues____Proteins destined for proteasome-mediated degradation may be ubiquitinated. Ubiquitination follows conjugation of ubiquitin to a conserved  cysteine residue of UBC homologues. This pathway functions in regulating many fundamental processes required for cell viability.TSG101 is one of several UBC homologues that lacks this active site cysteine.',
'UBQ' => 'SM00213____Ubiquitin homologues____Ubiquitin-mediated proteolysis is involved in the regulated turnover of  proteins required for controlling cell cycle progression ',
'UBX' => 'SM00166____Domain present in ubiquitin-regulatory proteins____Present in FAF1 and Shp1p.',
'UIM' => 'SM00726____Ubiquitin-interacting motif.____Present in proteasome subunit S5a and other ubiquitin-associated proteins. ',
'UME' => 'SM00802____Domain in UVSB PI-3 kinase, MEI-41 and ESR-1____Characteristic domain in UVSP PI-3 kinase, MEI-41 and ESR-1. Found in nucleolar proteins. Associated with FAT, FATC, PI3_PI4_kinase modules.',
'UTG' => 'SM00096____Uteroglobin____',
'UTRA' => 'SM00866________The UbiC transcription regulator-associated (UTRA) domain is a conserved ligand-binding domain that has a similar fold to PUBMED:12757941. It is believed to modulate activity of bacterial transcription factors in response to binding small molecules.',
'Ubox' => 'SM00504____Modified RING finger domain____Modified RING finger domain, without the full complement of Zn2+-binding ligands. Probable involvement in E2-dependent ubiquitination. ',
'VHP' => 'SM00153____Villin headpiece domain____',
'VHS' => 'SM00288____Domain present in VPS-27, Hrs and STAM____Unpublished observations. Domain of unknown function.',
'VIT' => 'SM00609____Vault protein Inter-alpha-Trypsin domain____',
'VKc' => 'SM00756________Family of likely enzymes that includes the catalytic subunit of vitamin K epoxide reductase. Bacterial homologues are fused to members of the thioredoxin family of oxidoreductases.',
'VPS10' => 'SM00602________',
'VPS9' => 'SM00167____Domain present in VPS9____Domain present in yeast vacuolar sorting protein 9 and other proteins.',
'VWA' => 'SM00327____von Willebrand factor (vWF) type A domain____VWA domains in extracellular eukaryotic proteins mediate adhesion via metal ion-dependent adhesion sites (MIDAS). Intracellular VWA domains and homologues in prokaryotes have recently been identified. The proposed VWA domains in integrin beta subunits have recently been substantiated using sequence-based methods.',
'VWC' => 'SM00214____von Willebrand factor (vWF) type C domain____',
'VWC_def' => 'SM00011________',
'VWC_out' => 'SM00215____von Willebrand factor (vWF) type C domain____',
'VWD' => 'SM00216____von Willebrand factor (vWF) type D domain____Von Willebrand factor contains several type D domains: D1 and D2 are present within the N-terminal propeptide whereas the remaining D domains are required for multimerisation.',
'WAP' => 'SM00217____Four-disulfide core domains____',
'WD40' => 'SM00320____WD40 repeats____Note that these repeats are permuted with respect to the structural repeats (blades) of the beta propeller domain.',
'WGR' => 'SM00773____Proposed nucleic acid binding domain____This domain is named after its most conserved central motif. It is found in a variety of polyA polymerases as well as in molybdate metabolism regulators (e.g. in E.coli) and other proteins of unknown function. The domain is found in isolation in some proteins and is between 70 and 80 residues in length. It is proposed that it may be a nucleic acid binding domain.',
'WH1' => 'SM00461____WASP homology region 1____Region of the Wiskott-Aldrich syndrome protein (WASp) that contains point mutations in the majority of patients with WAS. Unknown function. Ena-like WH1 domains bind polyproline-containing peptides, and that Homer contains a WH1 domain.',
'WH2' => 'SM00246____Wiskott Aldrich syndrome homology region 2____Wiskott Aldrich syndrome homology region 2 / actin-binding motif ',
'WHy' => 'SM00769____Water Stress and Hypersensitive response____',
'WIF' => 'SM00469____Wnt-inhibitory factor-1 like domain____Occurs as extracellular domain in metazoan Ryk receptor tyrosine kinases. C. elegans Ryk is required for cell-cuticle recognition. WIF-1 binds to Wnt and inhibits its activity.',
'WNT1' => 'SM00097____found in Wnt-1____',
'WR1' => 'SM00289____Worm-specific repeat type 1____Worm-specific repeat type 1. Cysteine-rich domain apparently unique (so far) to C. elegans. Often appears with KU domains. About 3 dozen worm proteins contain this domain.',
'WRKY' => 'SM00774____DNA binding domain____The WRKY domain is a DNA binding domain found in one or two copies in a superfamily of plant transcription factors. These transcription factors are involved in the regulation of various physiological programs that are unique to plants, including pathogen defense, senescence and trichome development. The domain is a 60 amino acid region that is defined by the conserved amino acid sequence WRKYGQK at its N-terminal end, together with a novel zinc-finger-like motif. It binds specifically to the DNA sequence motif (T)(T)TGAC(C/T), which is known as the W box. The invariant TGAC core is essential for function and WRKY binding.',
'WSC' => 'SM00321____present in yeast cell wall integrity and stress response component proteins____Domain present in WSC proteins, polycystin and fungal exoglucanase',
'WSN' => 'SM00453____Worm-specific (usually) N-terminal domain____',
'WW' => 'SM00456____Domain with 2 conserved Trp (W) residues____Also known as the WWP or rsp5 domain. Binds proline-rich polypeptides.',
'WWE' => 'SM00678____Domain in Deltex and TRIP12 homologues. Possibly involved in regulation of ubiquitin-mediated proteolysis.____',
'X8' => 'SM00768____Possibly involved in carbohydrate binding____The X8 domain, which may be involved in carbohydrate binding, is found in an Olive pollen antigen as well as at the C terminus of family 17 glycosyl hydrolases. It contains 6 conserved cysteine residues which presumably form three disulfide bridges.',
'XPGI' => 'SM00484____Xeroderma pigmentosum G I-region____domain in nucleases',
'XPGN' => 'SM00485____Xeroderma pigmentosum G N-region____domain in nucleases',
'XTALbg' => 'SM00247____Beta/gamma crystallins____Beta/gamma crystallins',
'YceI' => 'SM00867____YceI-like domain ____E. coli YceI is a base-induced periplasmic protein. The recent structure of a member of this family shows that it binds to polyisoprenoid. The structure consists of an extended, eight-stranded, antiparallel beta-barrel that resembles the lipocalin fold.',
'YqgFc' => 'SM00732____Likely ribonuclease with RNase H fold.____YqgF proteins are likely to function as an alternative to RuvC in most bacteria, and could be the principal holliday junction resolvases in low-GC Gram-positive bacteria. In Spt6p orthologues, the catalytic residues are substituted indicating that they lack enzymatic functions.',
'ZM' => 'SM00735____ZASP-like motif____Short motif (26 amino acids) present in an alpha-actinin-binding protein, ZASP, and similar molecules. ',
'ZP' => 'SM00241____Zona pellucida (ZP) domain____ZP proteins are responsible for sperm-adhesion fo the zona pellucida. ZP domains are also present in multidomain transmembrane proteins such as glycoprotein GP2, uromodulin and TGF-beta receptor type III (betaglycan).',
'ZU5' => 'SM00218____Domain present in ZO-1 and Unc5-like netrin receptors____Domain of unknown function.',
'Zalpha' => 'SM00550____Z-DNA-binding domain in adenosine deaminases.____Helix-turn-helix-containing domain. Also known as Zab.',
'ZipA_C' => 'SM00771____ZipA, C-terminal domain (FtsZ-binding)____C-terminal domain of ZipA, a component of cell division in E.coli. It interacts with the FtsZ protein in one of the initial steps of septum formation. The structure of this domain is composed of three alpha-helices and a beta-sheet consisting of six antiparallel beta-strands.',
'ZnF_A20' => 'SM00259____A20-like zinc fingers____A20- (an inhibitor of cell death)-like zinc fingers. The zinc finger mediates self-association in A20. These fingers also mediate IL-1-induced NF-kappaB activation.',
'ZnF_AN1' => 'SM00154____AN1-like Zinc finger____Zinc finger at the C-terminus of An1, a ubiquitin-like protein in Xenopus laevis.',
'ZnF_BED' => 'SM00614____BED zinc finger____DNA-binding domain in chromatin-boundary-element-binding proteins and transposases',
'ZnF_C2C2' => 'SM00440____C2C2 Zinc finger____Nucleic-acid-binding motif in transcriptional elongation factor TFIIS and RNA polymerases.',
'ZnF_C2H2' => 'SM00355____zinc finger____',
'ZnF_C2HC' => 'SM00343____zinc finger____',
'ZnF_C3H1' => 'SM00356____zinc finger____',
'ZnF_C4' => 'SM00399____c4 zinc finger in nuclear hormone receptors____',
'ZnF_CDGSH' => 'SM00704____CDGSH-type zinc finger. Function unknown.____',
'ZnF_CHCC' => 'SM00400____zinc finger____',
'ZnF_DBF' => 'SM00586____Zinc finger in DBF-like proteins____',
'ZnF_GATA' => 'SM00401____zinc finger binding to DNA consensus sequence [AT]GATA[AG] ____',
'ZnF_NFX' => 'SM00438________Repressor of transcription',
'ZnF_PMZ' => 'SM00575____plant mutator transposase zinc finger____',
'ZnF_RBZ' => 'SM00547____Zinc finger domain____Zinc finger domain in Ran-binding proteins (RanBPs), and other proteins. In RanBPs, this domain binds RanGDP.',
'ZnF_Rad18' => 'SM00734____Rad18-like CCHC zinc finger____Yeast Rad18p functions with Rad5p in error-free post-replicative DNA repair. This zinc finger is likely to bind nucleic-acids.',
'ZnF_TAZ' => 'SM00551____TAZ zinc finger, present in p300 and CBP ____',
'ZnF_TTF' => 'SM00597________zinc finger in transposases and transcription factors',
'ZnF_U1' => 'SM00451____U1-like zinc finger____Family of C2H2-type zinc fingers, present in matrin, U1 small nuclear ribonucleoprotein C and other RNA-binding proteins. ',
'ZnF_UBP' => 'SM00290____Ubiquitin Carboxyl-terminal Hydrolase-like zinc finger____',
'ZnF_UBR1' => 'SM00396____Putative zinc finger in N-recognin, a recognition component of the N-end rule pathway____Domain is involved in recognition of N-end rule substrates in yeast Ubr1p',
'ZnF_ZZ' => 'SM00291____Zinc-binding domain, present in Dystrophin, CREB-binding protein.____Putative zinc-binding domain present in dystrophin-like proteins,  and CREB-binding protein/p300 homologues. The ZZ in dystrophin appears to bind calmodulin. A missense mutation of one of the conserved cysteines in dystrophin results in a patient with Duchenne muscular dystrophy [3].',
'ZnMc' => 'SM00235____Zinc-dependent metalloprotease____Neutral zinc metallopeptidases. This alignment represents a subset of known subfamilies. Highest similarity occurs in the HExxH zinc-binding site/ active site.',
'Zn_dep_PLPC' => 'SM00770____Zinc dependent phospholipase C (alpha toxin)____This domain conveys a zinc dependent phospholipase C activity (EC 3.1.4.3). It is found in a monomeric phospholipase C of Bacillus cereus as well as in the alpha toxin of Clostridium perfringens and Clostridium bifermentans, which is involved in haemolysis and cell rupture. It is also found in a lecithinase of Listeria monocytogenes, which is involved in breaking the 2-membrane vacuoles that surround the bacterium. Structure information: PDB 1ca1.',
'Zn_pept' => 'SM00631________',
'Zpr1' => 'SM00709____Duplicated domain in the epidermal growth factor- and elongation factor-1alpha-binding protein Zpr1. Also present in archaeal proteins.____',
'acidPPc' => 'SM00014____Acid phosphatase homologues____',
'alkPPc' => 'SM00098____Alkaline phosphatase homologues____',
'btg1' => 'SM00099____tob/btg1 family____The tob/btg1 is a family of proteins that inhibit cell proliferation. ',
'cNMP' => 'SM00100____Cyclic nucleotide-monophosphate binding domain____Catabolite gene activator protein (CAP) is a prokaryotic homologue of eukaryotic cNMP-binding domains, present in ion channels, and  cNMP-dependent kinases.',
'calpain_III' => 'SM00720________',
'dDENN' => 'SM00801____Domain always found downstream of DENN domain, found in a variety of signalling proteins____The dDENN domain is part of the tripartite DENN domain. It is always found downstream of the DENN domain itself, which is found in a variety of signalling proteins involved in Rab-mediated processes or regulation of MAPKs signalling pathways. The DENN domain is always encircled on both sides by more divergent domains, called uDENN (for upstream DENN) and dDENN (for downstream DENN). The function of the DENN domain remains to date unclear, although it appears to represent a good candidate for a GTP/GDP exchange activity.',
'eIF1a' => 'SM00652____eukaryotic translation initiation factor 1A____',
'eIF2B_5' => 'SM00653____domain present in translation initiation factor eIF2B and eIF5____',
'eIF5C' => 'SM00515____Domain at the C-termini of GCD6, eIF-2B epsilon, eIF-4 gamma and eIF-5____',
'eIF6' => 'SM00654____translation initiation factor 6____',
'fCBD' => 'SM00236____Fungal-type cellulose-binding domain____Small four-cysteine cellulose-binding domain of fungi',
'rADc' => 'SM00650____Ribosomal RNA adenine dimethylases____',
'small_GTPase' => 'SM00010____Small GTPase of the Ras superfamily; ill-defined subfamily____SMART predicts Ras-like small GTPases of the ARF, RAB, RAN, RAS, and SAR subfamilies. Others that could not be classified in this way are predicted to be members of the small GTPase superfamily without predictions of the subfamily.',
'tRNA_SAD' => 'SM00863____Threonyl and Alanyl tRNA synthetase second additional domain____The catalytically active form of threonyl/alanyl tRNA synthetase is a dimer. Within the tRNA synthetase class II dimer, the bound tRNA interacts with both monomers making specific interactions with the catalytic domain, the C-terminal domain, and this SAD domain (the second additional domain). The second additional domain is comprised of a pair of perpendicularly orientated antiparallel beta sheets, of four and three strands, respectively, that surround a central alpha helix that forms the core of the domain.',
't_SNARE' => 'SM00397____Helical region found in SNAREs____All alpha-helical motifs that form twisted and parallel four-helix bundles in target soluble N-ethylmaleimide-sensitive factor (NSF) attachment protein (SNAP) receptor proteins. This motif found in "Q-SNAREs". ',
'uDENN' => 'SM00800____Domain always found upstream of DENN domain, found in a variety of signalling proteins____The uDENN domain is part of the tripartite DENN domain. It is always found upstream of the DENN domain itself, which is found in a variety of signalling proteins involved in Rab-mediated processes or regulation of MAPKs signalling pathways. The DENN domain is always encircled on both sides by more divergent domains, called uDENN (for upstream DENN) and dDENN (for downstream DENN). The function of the DENN domain remains to date unclear, although it appears to represent a good candidate for a GTP/GDP exchange activity.',
'zf-AD' => 'SM00868____Zinc-finger associated domain (zf-AD)____The zf-AD domain, also known as ZAD, forms an atypical treble-cleft-like zinc co-ordinating fold. The zf-AD domain is thought to be involved in mediating dimer formation, but does not bind to DNA.');

	$start_description_str = $smart_description_hash{$domain_name};
	return $start_description_str;
} # sub get_smart_description {

1;
